---
title: 'Useful bioinformatics software, tools, pipelines and workflows'
subtitle: 'Some of the million tools that are out there'
date: 2018-06-30 00:00:00
featured_image: '/images/miscellaneous/tools.jpg'
---

![](/mysite/images/miscellaneous/tools.jpg)

Every now and then, I come across papers describing new tools and software (mainly) for some kind of metagenomics, amplicon (aka metabarcoding), genome related analysis and other things that interest me. Even though I might not need them at that particular moment, I make a note in case I would like to use them in a future project. 

So, without further ado, here is what I have gathered so far. Of course, some of the tools may not be maintained anymore. And, it goes without saying, this is a non-exhaustive list of software; for sure there are twice as more out there that I have missed. Also, this list doesn't include tools that I have been part of their development (one way or the other), such as [PEMA](https://hariszaf.github.io/pema_documentation/), [DARN](https://hariszaf.github.io/darn/) and [metaGOflow](https://metagoflow.readthedocs.io/en/latest/). 



| Software                 | Aim        | Doi | Source code/Website |
|----------------------|---------------|------------------|-------|
| RiboViz | analyzing and visualizing ribosome profiling datasets | 10.1186/s12859-017-1873-8 | https://github.com/shahpr/RiboViz |
| GAIA 2.0 | metagenomics analysis |  | https://www.gaia.sequentiabiotech.com/ |
| BURRITO  | interactive visualization of the links between taxonomic composition and function in microbiome datasets | 10.3389/fmicb.2018.00365 | https://github.com/borenstein-lab/burrito |
| paprica | metabolic inference from 16S rRNA gene sequence libraries / PAthway PRediction by phylogenetIC plAcement | 10.1371/journal.pone.0135868 | https://github.com/bowmanjeffs/paprica |
| LotuS  | lightweight complete 16S pipeline | 10.1186/2049-2618-2-30 | http://psbweb05.psb.ugent.be/lotus/ |
| BioSurfDB | biological compounds and mechanisms involved in biosurfactant production and/or biodegradation  | 10.1093/database/bav033 | http://www.biosurfdb.org/ |
| metabolic-hmms | HMM databases for microbial metabolic genes of environmental/biogeochemical importance |  | https://github.com/banfieldlab/metabolic-hmms |
| T-Coffee Server | collection of tools for Computing, Evaluating and Manipulating Multiple Alignments of DNA, RNA, Protein Sequences and Structures | 10.1006/jmbi.2000.4042 | http://www.tcoffee.org/ |
| FWMFG (Freshwater microbial field guide) | freshwater 16S rRNA gene sequence database | 10.1128/MMBR.00028-10 | https://github.com/mcmahon-uw/FWMFG |
| TaxAss  | Taxonomy Assignment workflow for 16S datasets | 10.1101/214288 | https://github.com/McMahonLab/TaxAss |
| Langille Lab PICRUSt Galaxy Instance (Correct for copy number variation) | Galaxy version Langille Lab |  | http://galaxy.morganlangille.com/ |
| Hutlab | Galaxy version Huttenhower Lab  |  | http://huttenhower.sph.harvard.edu/galaxy/root |
| microPITA (Picking Interesting Taxonomic Abundance) | computational tool enabling sample selection in two-stage (tiered) studies | 10.1038/ismej.2013.139 | http://huttenhower.sph.harvard.edu/micropita |
| CIPRES Science Gateway | inference of large phylogenetic trees |  | https://www.phylo.org/ |
| robustness | microbial community taxa-function robustness estimation pipeline |  | https://github.com/borenstein-lab/robustness |
| Phinch | interactive, exploratory data visualization framework for –Omic datasets | 10.1101/009944 | http://phinch.org/index.html |
| ETE Toolkit | Phylogenetic tree (newick) viewer | 10.1093/molbev/msw046 | http://etetoolkit.org/treeview/ |
| seqenv | Assign environment ontology (EnvO) terms to short DNA sequences | 10.7717/peerj.2690 | https://github.com/xapple/seqenv |
| SparCC | computing correlations in compositional data (16S, metagenomics, etc'). | 10.1371/journal.pcbi.1002687 | https://bitbucket.org/yonatanf/sparcc |
| micca | MICrobial Community Analysis | 10.1038/srep09743 | http://micca.org/ |
| Perl-Bioinformatics-Scripts | Perl-Bioinformatics-Scripts |  | https://github.com/BryanChim/Perl-Bioinformatics-Scripts |
| Perl Scripts for bioinformatics | Perl Scripts for bioinformatics |  | http://bioinforma.weebly.com/perl-for-bioinformatics.html#/ |
| FaBox  | online fasta sequence toolbox |  | http://users-birc.au.dk/biopv/php/fabox/ |
| 16S rRNA Pyrosequencing Process Utilities | 16S rRNA Pyrosequencing Process Utilities |  | http://alrlab.research.pdx.edu/aquificales/bioinformatics_scripts.html |
| microDB | database that aims to provide the most complete census to-date of the environmental distribution of prokaryotes | 10.1111/j.1758-2229.2009.00030.x | http://botero.cnb.csic.es/envDB/ |
| Volcano Plot | Volcano Plot |  | https://paolo.shinyapps.io/ShinyVolcanoPlot/ |
| copynum_estimation | Software for copy number and organismal abundance estimation | 10.1371/journal.pcbi.1002743 | http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002743#s5 |
| R script to determine the minimum number of sequences required for reliable microbial community analysis | A novel method to determine the minimum number of sequences | 10.1016/j.mimet.2017.06.006 | https://www.sciencedirect.com/science/article/pii/S0167701217301562 |
| CopyRighter | improving the accuracy of microbial community profiles through lineage-specific gene copy number correction | 10.1186/2049-2618-2-11 | https://github.com/fangly/AmpliCopyrighter/ |
| rrnDB | interpreting rRNA gene abundance in bacteria and archaea | 10.1093/nar/gku1201 | https://rrndb.umms.med.umich.edu/ |
| mockrobiota | a public resource for sharing, validating, and documenting mock community data resources | 10.1128/mSystems.00062-16 | http://caporaso-lab.github.io/mockrobiota/ |
| Metadex | The Metagenomic Data Explorer |  | https://search.proquest.com/openview/3e734d930740f85542e4bd14ee1ca762/1?pq-origsite=gscholar&cbl=18750&diss=y |
| TAXXI | Taxonomy cross-validation by identity | 10.7717/peerj.4652 | https://github.com/rcedgar/taxxi |
| NGLess | NGS Processing with Less Work | 10.1186/s40168-019-0684-8 | https://ngless.embl.de/ |
| MASAME | MASAME | 10.1111/1574-6941.12437 | https://github.com/pbuttigieg/masame |
| MicrO | A Microbial Ontology of Phenotypic and Metabolic Characters, Assays, and Culture Media Found in Prokaryotic Taxonomic Descriptions |  | https://github.com/pbuttigieg/MicrO |
| MicroPIE  | Microbial Phenomics Information Extractor | 10.1186/s12859-016-1396-8 | https://github.com/biosemantics/micropie2 |
| PrimerProspector | de novo design and taxonomic analysis of barcoded polymerase chain reaction primers | 10.1093/bioinformatics/btr087 | http://pprospector.sourceforge.net/ |
| tax2tree | automatically decorating taxonomy onto a phylogenetic tree | 10.1038/ismej.2011.139 | http://tax2tree.sourceforge.net |
| SitePainter | interactive graphical tool that allows investigators to create or upload pictures of their study site, load diversity analyses, and display both diversity and taxonomy results in a spatial context | 10.1093/bioinformatics/btr685 | http://biocore.github.io/SitePainter/ |
| PATRIC  | integrated data and analysis tools to support biomedical research on bacterial infectious diseases | 10.1093/nar/gkw1017 | https://www.patricbrc.org/ |
| NINJA-OPS | Not Just Another - OTU Picking Solution | 10.1371/journal.pcbi.1004658 | https://github.com/GabeAl/NINJA-OPS |
| BugBase | measuring high-level phenotypes in your microbiome | 10.1101/133462 | https://bugbase.cs.umn.edu/index.html |
| DAS | Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy | 10.1038/s41564-018-0171-1 | https://github.com/cmks/DAS_Tool |
| WHAM! (Workflow Hub for Automated Metagenomic Exploration) | a web-based visualization suite for user-defined analysis of metagenomic shotgun sequencing data | 10.1186/s12864-018-4870-z | https://github.com/ruggleslab/jukebox/tree/master/wham_v1 |
| SqueezeM | metagenomic analysis pipeline from reads tobins | 10.1101/347559 |  https://github.com/jtamames/SqueezeM |
| SAMSA2 | standalone metatranscriptome analysis pipeline | 10.1186/s12859-018-2189-z | https://github.com/transcript/samsa2 |
| BlastKOALA | Automatic annotation and KEGG mapping server suitable for functional characterization of complete genomes | 10.1016/j.jmb.2015.11.006 | https://www.kegg.jp/blastkoala/ |
| GhostKOALA  | Automatic annotation and KEGG mapping server suitable for functional characterization of metagenomes | 10.1016/j.jmb.2015.11.006 | https://www.kegg.jp/ghostkoala/ |
| PAIPline | Pathogen identification in metagenomic and clinical next generation sequencing samples |  | https://gitlab.com/andreas.andrusch/paipline |
| Carnelian | Enhanced functional profiling of whole metagenome sequencing reads | 10.1101/375121 | http://cb.csail.mit.edu/cb/carnelian/ |
| DAFGA | Diversity analysis of functional gene amplicons | 10.1093/bioinformatics/btu394 | https://github.com/outbig/DAFGA |
| COGNIZER | A Framework for Functional Annotation of Metagenomic Datasets | 10.1371/journal.pone.0142102 | http://metagenomics.atc.tcs.com/cognizer, https://metagenomics.atc.tcs.com/function/cognizer |
| WebMGA | a customizable web server for fast metagenomic sequence analysis | 10.1186/1471-2164-12-444 | http://weizhongli-lab.org/metagenomic-analysis |
| gcMeta | a Global Catalogue of Metagenomics platform to support the archiving, standardization and analysis of microbiome data | 10.1093/nar/gky1008 | https://gcmeta.wdcm.org/ |
| CheckM | assessing the quality of genomes recovered from isolates, single cells, or metagenomes | 10.1101/gr.186072.114 | http://ecogenomics.github.io/CheckM/ |
| dRep | fast and accurate genomic comparisons | 10.1038/ismej.2017.126 | https://github.com/MrOlm/drep |
| FLASH | Fast Length Adjustment of SHort reads | 10.1093/bioinformatics/btr507 | https://ccb.jhu.edu/software/FLASH/ |
| Btrim | adapter and quality trimming | 10.1016/j.ygeno.2011.05.009 | http://graphics.med.yale.edu/trim/ |
| SPIEC-EASI | Sparse and Compositionally Robust Inference of Microbial Ecological Networks | 10.1371/journal.pcbi.1004226 | https://github.com/zdk123/SpiecEasi |
| OTUX | V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes | 10.1093/dnares/dsy045 | https://web.rniapps.net/otux/ |
| iVikodak | A Platform and Standard Workflow for Inferring, Analyzing, Comparing, and Visualizing the Functional Potential of Microbial Communities | 10.3389/fmicb.2018.03336 | https://web.rniapps.net/iVikodak/ |
| ranacapa | eDNA Data exploration |  | https://gauravsk.shinyapps.io/ranacapa/ |
| ANCOM | Analysis of Composition of Microbiomes | 10.3402/mehd.v26.27663 | https://sites.google.com/site/siddharthamandal1985/research |
| MNEMONIC  | MetageNomic Experiment Mining to create an OTU Network of Inhabitant Correlations | 10.1186/s12859-019-2623-x | https://gitlab.com/wrenlab/mnemonic |
| FENNEC  | Functional Exploration of Natural Networks and Ecological Communities | 10.1111/2041-210X.13060 | https://fennec.molecular.eco/default_data/startpage |
| FunGuild | An open annotation tool for parsing fungal community datasets by ecological guild | 10.1016/j.funeco.2015.06.006 | https://github.com/UMNFuN/FUNGuild |
| FAPROTAX | Functional Annotation of Prokaryotic Taxa | 10.1126/science.aaf4507 | http://www.loucalab.com/archive/FAPROTAX/ |
| Bcdatabaser | on-the-fly reference database creation for (meta-)barcoding | 10.1093/bioinformatics/btz960 | https://github.com/molbiodiv/bcdatabaser |
| SLIM  | A web app for environmental DNA metabarcoding analysis | 10.1186/s12859-019-2663-2 | https://trtcrd.github.io/SLIM/ |
| PIPITS | An automated pipeline for analyses of fungal internal transcribed spacer (ITS) sequences from the Illumina sequencing platform  | 10.1111/2041-210X.12399 | https://github.com/hsgweon/pipits |
| mOTUs2 | profile a metagenomic or metatranscriptomic sample | 10.1038/s41467-019-08844-4 | https://motu-tool.org/ |
| Cascabel | pipeline designed to run amplicon sequence analysis | 10.1101/809384 | https://github.com/AlejandroAb/CASCABEL |
| proGenomes3 | improved database for accurate and consistent habitat, taxonomic and functional annotations of prokaryotic genomes | 10.1093/nar/gkz1002 | http://progenomes.embl.de/ |
| Kelpie  | Generating full-length 'amplicons' from whole-metagenome datasets | 10.7717/peerj.6174 | https://github.com/PaulGreenfieldOz/WorkingDogs/tree/master/Kelpie |
| PhenDB  | Prediction of bacterial phenotypes | 10.1186/1471-2105-16-S14-S1 | https://phendb.csb.univie.ac.at/ |
| phyloFlash | Rapid SSU rRNA profiling and targeted assembly from metagenomes | 10.1101/521922 | https://github.com/HRGV/phyloFlash |
| Shade_CurrOpinMicro | Containing script for investigating core microbiota | 10.1016/j.mib.2019.09.008 | https://github.com/ShadeLab/PAPER_Shade_CurrOpinMicro |
| PMBD | a Comprehensive Plastics Microbial Biodegradation Database | 10.1093/database/baz119 | http://pmbd.genome-mining.cn/home/ |
| MAR databases | a collection of richly annotated and manually curated contextual (metadata) and sequence databases | 10.1093/nar/gkx1036 | https://mmp.sfb.uit.no/databases/ |
| iMicrobe | Tools and data-driven discovery platform for the microbiome sciences | 10.1093/gigascience/giz083 | https://www.imicrobe.us/ |
| MelonnPan   | Predictive metabolomic profiling of microbial communities using amplicon or metagenomic sequences | 10.1038/s41467-019-10927-1 | http://huttenhower.sph.harvard.edu/melonnpan |
|Melon|metagenomic long-read-based taxonomic identification and quantification using marker genes|10.1186/s13059-024-03363-y|https://github.com/xinehc/melon|
| FeGenie | Tool for the Identification of Iron Genes and Iron Gene Neighborhoods in Genome and Metagenome Assemblies | 10.3389/fmicb.2020.00037 | https://github.com/Arkadiy-Garber/FeGenie |
| NG-Tax | pipeline for 16S rRNA gene amplicon sequencing data | 10.12688/f1000research.9227.2 | https://github.com/JavierRamiroGarcia/NG-Tax |
| MetaShot | taxon classification of host-associated microbiome from shotgun metagenomic data | 10.1093/bioinformatics/btx036 | https://github.com/bfosso/MetaShot |
| LEMMI | A Live Evaluation of Computational Methods for Metagenome Investigation | 10.1101/507731 | https://lemmi.ezlab.org/ |
| proGenomes2  | annotated bacterial and archaeal genomes | 10.1093/nar/gkz1002 | http://progenomes.embl.de/ |
| SOBA | sequence ontology bioinformatics analysis | 10.1093/nar/gkq426 | http://www.sequenceontology.org/cgi-bin/soba.cgi |
| CARD 2020 | antibiotic resistance database | 10.1093/nar/gkz935 | https://card.mcmaster.ca |
| MEGARes | Antimicrobial Database for High-Throughput Sequencing | 10.1093/nar/gkw1009 | https://megares.meglab.org/ |
| Metazen | metadata capture for metagenomes | 10.1186/1944-3277-9-18 | https://github.com/MG-RAST/metazen |
| Lathe | A tool for generating bacterial genomes from metagenomes with nanopore long read sequencing | 10.1038/s41587-020-0422-6 | https://github.com/bhattlab/lathe |
| Metagenomics Workflows | Workflows for metagenomic sequence data processing and analysis | 10.1038/s41587-020-0422-6 | https://github.com/bhattlab/metagenomics_workflows |
| StrainSifter | bioinformatic pipeline for detecting the presence of a bacterial strain in one or more metagenome(s) |  | https://github.com/bhattlab/StrainSifter |
| MicrobiomeBestPracticeReview | Current Challenges and Best Practice Protocols for Microbiome Analysis using Amplicon and Metagenomic Sequencing | 10.1093/bib/bbz155 | https://github.com/grimmlab/MicrobiomeBestPracticeReview |
| BacPipe | whole genome sequencing pipeline for clinical diagnostic bacteriology and outbreak detection | 10.1016/j.isci.2019.100769 | https://github.com/wholeGenomeSequencingAnalysisPipeline/BacPipe |
| Jovian  | Metagenomics/viromics pipeline |  | https://github.com/DennisSchmitz/Jovian |
| Evergreen  | Large scale automated phylogenomic analysis of bacterial whole-genome isolates | 10.1101/540138 | https://cge.cbs.dtu.dk/services/Evergreen |
| RIEMS | Reliable Information Extraction from Metagenomic Sequence datasets |  | https://github.com/EBI-COMMUNITY/fli-RIEMS |
| PIA (Phylogenetic Intersection Analysis) | Taxonomic Assignment of Metagenomic Data | 10.3389/fevo.2020.00084 | https://github.com/Allaby-lab/PIA |
| PhyloPhlAn | Precise phylogenetic analysis of microbial isolates and genomes from metagenomes |  | https://github.com/biobakery/phylophlan |
| Link-HD | an R package to integrate multiple heterogeneous datasets based on STATIS / a versatile framework to explore and integrate heterogeneous microbial communities | 10.1093/bioinformatics/btz862 | https://lauzingaretti.github.io/LinkHD-examples/ |
| MicrobiomeAnalyst | comprehensive statistical, visual and meta-analysis of microbiome data | 10.1093/nar/gkx295 ; 10.1038/s41596-019-0264-1 | https://www.microbiomeanalyst.ca/MicrobiomeAnalyst/home.xhtml |
| decontam | Identifying contaminants in marker-gene and metagenomics data | 10.1186/s40168-018-0605-2 | https://benjjneb.github.io/decontam/vignettes/decontam_intro.html |
| GMPR | Calculate the normalizing factors for zeroinflated sequencing data such as microbiome sequencing data |  | https://github.com/jchen1981/GMPR |
| MetaLonDA | METAgenomic LONgitudinal Differential Abundance method | 10.1186/s40168-018-0402-y | https://cran.r-project.org/web/packages/MetaLonDA/vignettes/MetaLonDA.html |
| MetaBAT 2 | an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | 10.7717/peerj.7359 | https://bitbucket.org/berkeleylab/metabat/src/master/ |
| FunGeCo  | web based tool for estimation of Functional potential of bacterial genomes and microbiomes using Gene Context information | 10.1093/bioinformatics/btz957 | https://web.rniapps.net/fungeco/ |
| TAX CREdiT data | TAXonomic ClassifieR Evaluation Tool data repository | 10.1186/s40168-018-0470-z | https://github.com/caporaso-lab/tax-credit-data |
| Metagenome-Assembled Genomes Orchestra (MAGO) | computational framework for high-quality production and large-scale evolutionary analysis of metagenome assembled genomes | 10.1093/molbev/msz237 | http://mago.fe.uni-lj.si/ |
| RFPlasmid | Predicting plasmid contigs from assemblies using single copy marker genes, plasmid genes, kmers |  | https://github.com/aldertzomer/RFPlasmid |
| Phage_tools |  |  | https://github.com/sxh1136/Phage_tools |
| GlobalFungi | Global database of fungal records from high-throughput-sequencing metabarcoding studies | 10.1101/2020.04.24.060384 | http://globalfungi.com/ |
| CAMITAX | Taxon labels for microbial genomes | 10.1093/gigascience/giz154 | https://github.com/CAMI-challenge/CAMITAX |
| AutoTax | Comprehensive ecosystem-specific 16S rRNA gene databases with automated taxonomy assignment provide species-level resolution | 10.1101/672873 | https://github.com/KasperSkytte/AutoTax |
| fSSU | Retrieval of a million high-quality, full-length microbial 16S and 18S rRNA gene sequences without primer bias | 10.1038/nbt.4045 | https://github.com/MadsAlbertsen/fSSU |
| barrnap | BAsic Rapid Ribosomal RNA Predictor |  | https://github.com/tseemann/barrnap |
| MicrobiomeAnalyst  | comprehensive statistical, functional, and meta-analysis of microbiome data | 10.1038/s41596-019-0264-1 | https://www.microbiomeanalyst.ca/MicrobiomeAnalyst/ |
| Ocean Gene Atlas | exploring the biogeography of plankton genes online | 10.1093/nar/gky376 | http://tara-oceans.mio.osupytheas.fr/ocean-gene-atlas/ |
| SHOGUN | modular, accurate, and scalable framework for microbiome quantification | 10.1093/bioinformatics/btaa277 | https://github.com/knights-lab/SHOGUN |
| MetaDEGalaxy | Galaxy workflow for differential abundance analysis of 16s metagenomic data | 10.12688/f1000research.18866.2 |  |
| MACADAM database | MetAboliC pAthways DAtabase for Microbial taxonomic groups | 10.1093/database/baz049 | https://github.com/maloleboulch/MACADAM-Database |
| CAMISIM | simulating metagenomes and microbial communities | 10.1186/s40168-019-0633-6 | https://github.com/CAMI-challenge/CAMISIM |
| PhyloPythiaS+ | rapid reconstruction of low-ranking taxonomic bins from metagenomes | 10.7717/peerj.1603 | https://github.com/algbioi/ppsp/wiki |
| MBGD | Microbial Genome Database | 10.1093/nar/gky1054 | http://mbgd.genome.ad.jp/ |
| FVE-novel | Recovering Draft Genomes of Novel Viruses and Phages in Metagenomic Data | 10.21203/rs.3.rs-17154/v1 | https://github.com/saima-tithi/FVE-novel |
| VirFinder | identifying viral sequences from assembled metagenomic data | 10.1186/s40168-017-0283-5 | https://github.com/jessieren/VirFinder |
| Prokaryotic Virus Orthologous Groups (pVOGs) | a resource for comparative genomics and protein family annotation | 10.1093/nar/gkw975 | http://dmk-brain.ecn.uiowa.edu/pVOGs/ |
| VirSorter | mining viral signal from microbial genomic data | 10.7717/peerj.985 | https://github.com/simroux/VirSorter |
| Prodigal (PROkaryotic DYnamic programming Gene-finding ALgorithm) | prokaryotic gene recognition and translation initiation site identification | 10.1186/1471-2105-11-119 | http://compbio.ornl.gov/prodigal/ |
| TaxonTableTools  | graphical user interface software to explore and visualise DNA metabarcoding data | 10.1101/2020.08.24.264317 | https://github.com/TillMacher/TaxonTableTools |
| VirHostMatcher-Net | A network-based integrated framework for predicting virus–prokaryote interactions | 10.1093/nargab/lqaa044 | https://github.com/WeiliWw/VirHostMatcher-Net |
| FROGS | Find, Rapidly, OTUs with Galaxy Solution | 10.1093/bioinformatics/btx791 | https://github.com/geraldinepascal/FROGS.git |
| ShotMAP | Automated and Accurate Estimation of Gene Family Abundance from Shotgun Metagenomes | 10.1371/journal.pcbi.1004573 | https://github.com/sharpton/shotmap |
| STRONG | STrain Resolution ON assembly Graph | 10.1101/2020.09.06.284828 | https://github.com/chrisquince/STRONG |
| Cviewer | A Java-based statistical framework for integration of shotgun metagenomics with other omics technologies | 10.1186/s40168-024-01834-9 | https://github.com/KociOrges/cviewer |
| CONCOCT | unsupervised binning of metagenomic contigs by using nucleotide composition, coverage data in multiple samples and linkage data from paired end reads | 10.1038/nmeth.3103 | https://github.com/BinPro/CONCOCT |
| Prokka | rapid prokaryotic genome annotation | 10.1093/bioinformatics/btu153 | https://github.com/tseemann/prokka |
| CALOUR | exploratory and interactive microbiome analysis based on heatmaps | 10.1128/mSystems.00269-18 | https://github.com/biocore/calour |
| CAMITAX | Taxon labels for microbial genomes | 10.1093/gigascience/giz154 | https://github.com/CAMI-challenge/CAMITAX |
| BIOCOM-PIPE | characterization of microbial diversity from 16S, 18S and 23S rRNA gene amplicons | 10.1186/s12859-020-03829-3 | https://doi.org/10.5281/zenodo.3678129 |
| PathoFact | prediction of virulence factors and antimicrobial resistance genes in metagenomic data | 10.1101/2020.03.24.006148 | https://git-r3lab.uni.lu/laura.denies/PathoFact |
| ideel | quick test for interrupted ORFs in bacterial/microbial genomes |  | https://github.com/mw55309/ideel |
| SCATA | Sequence Clustering and Analysis of Tagged Amplicons | 10.1139/cjm-2017-0099 | https://scata.mykopat.slu.se/ |
| Ananke | Clustering marker gene data by time-series patterns | 10.7717/peerj.3812 | https://github.com/beiko-lab/ananke |
| phylo­Flash | rapid small-subunit rRNA profiling and targeted assembly from metagenomes | 10.1128/mSystems.00920-20 | https://github.com/HRGV/phyloFlash |
| bioBakery 3 | Integrating taxonomic, functional, and strain-level profiling  of diverse microbial communities | 10.1101/2020.11.19.388223 | https://github.com/biobakery/biobakery/wiki , http://segatalab.cibio.unitn.it/tools/biobakery/ |
| Tax4Fun2 | prediction of habitat-specific functional profiles and functional redundancy based on 16S rRNA gene sequences | 10.1186/s40793-020-00358-7 | https://github.com/bwemheu/Tax4Fun2 |
| Docker pipelines | a collection of nextflow workflows related to nanopore data, taxonomy and antibiotic resistance | 10.5281/zenodo.4030719 | https://github.com/replikation/docker_pipelines |
| What_the_Phage | A scalable workflow for the identification and analysis of phage sequences | 10.1101/2020.07.24.219899 | https://github.com/replikation/What_the_Phage |
| Cascabel | Scalable and Versatile Amplicon Sequence Data Analysis Pipeline | 10.3389/fgene.2020.489357 | https://github.com/AlejandroAb/CASCABEL |
| CoMA | an intuitive and user-friendly pipeline for amplicon-sequencing data analysis | 10.1371/journal.pone.0243241 | https://www.uibk.ac.at/microbiology/services/coma.html |
| MARES | a replicable pipeline and curated reference database for marine eukaryote metabarcoding | 10.1038/s41597-020-0549-9 | https://github.com/wpearman1996/MARES_database_pipeline |
| LULU | A r-package for distribution based post clustering curation of amplicon data | 10.1038/s41467-017-01312-x | https://github.com/tobiasgf/lulu |
| fusionDB | assessing microbial diversity and environmental preferences via functional similarity networks | 10.1093/nar/gkx1060 | https://services.bromberglab.org/fusiondb/ |
| Vamb | Improved metagenome binning and assembly using deep variational autoencoders | 10.1038/s41587-020-00777-4 | https://github.com/RasmussenLab/vamb |
| eukaryotesV4 database | curated database of V4 region of 18S rDNA. It was built to extract and taxonomically classify short metagenomic reads (mTags) | 10.1111/1755-0998.13147 | https://github.com/aleixop/eukaryotesV4 |
| EcolUtils | Utilities for community ecology analysis |  | https://github.com/GuillemSalazar/EcolUtils |
| Recurrence-Index | the R code for calculating the index |  | https://github.com/CaterinaRG/Recurrence-Index |
| DEMENTpy | A trait and individual based spatially explicit soil microbial systems modelling framework | 10.1111/j.1461-0248.2012.01807.x | https://github.com/bioatmosphere/DEMENTpy |
| iDynoMiCS | Individual-based Dynamics of Microbial Communities Simuator | 10.1111/j.1462-2920.2011.02414.x | https://www.birmingham.ac.uk/generic/idynomics/index.aspx |
| NUFEB | modelling and simulating microbial communities |  | https://github.com/nufeb/NUFEB |
| MicroMap | web application that can be used to map global microbial abundance |  | https://micromap.icm.csic.es/ |
| scrapp | A tool to assess the diversity of microbial samples from phylogenetic placements | 10.1111/1755-0998.13255 | https://github.com/pbdas/scrapp |
| mgPipe | pipeline to integrate microbial abundances (coming from metagenomic data) with constraint-based modeling, creating individuals' personalized models |  | https://github.com/opencobra/cobratoolbox/tree/master/src/analysis/multiSpecies/microbiomeModelingToolbox/mgPipe |
| MIGA | Microbial Genomes Atlas Online | 10.1093/nar/gky467 | http://microbial-genomes.org/ |
| enveomics collection | a toolbox for specialized analyses of microbial genomes and metagenomes | 10.7287/peerj.preprints.1900v1 | http://enve-omics.ce.gatech.edu/ |
| PATO | Pangenome Analysis Toolkit | 10.1101/2021.01.30.428878 | https://github.com/irycisBioinfo/PATO |
| Protologger | all-in-one genome description tool, aimed at simplifying the process of gathering the data required for writing protologues |  | http://protologger.de/ |
| MetaStorm | metagenomic analysis server | 10.1371/journal.pone.0162442 | http://bench.cs.vt.edu/MetaStorm/ |
| NanoARG  | Antimicrobial Resistance analysis for nanopore reads | 10.1101/483248 | http://bench.cs.vt.edu/nanoARG |
| ARGminer | Antibiotic Resistance Gene miner databasε |  | https://bench.cs.vt.edu/argminer |
| DeepARG | a deep learning approach for predicting antibiotic resistance genes from metagenomic data | 10.1186/s40168-018-0401-z | https://bench.cs.vt.edu/deeparg_analyze/ |
| eDNAFlow  | automated, reproducible and scalable workflow for analysis of environmental DNA (eDNA) sequences exploiting Nextflow and Singularity | 10.1111/1755-0998.13356 | https://github.com/mahsa-mousavi/eDNAFlow |
| refdb | A reference database manager for R |  | https://github.com/fkeck/refdb |
| bioseq | A Toolbox for Manipulating Biological Sequences in R | 10.1111/2041-210X.13490 | https://github.com/fkeck/bioseq |
| GAPeDNA  | Assessing and mapping global species gaps in genetic databases for eDNA metabarcoding | 10.1111/ddi.13142 | https://shiny.cefe.cnrs.fr/GAPeDNA/ |
| AutoTax | automatically generates de novo taxonomy from full length 16S rRNA amplicon sequence variants (FL-ASVs) | 10.1128/mBio.01557-20 | https://github.com/KasperSkytte/AutoTax |
| Decona  | From demultiplexing to consensus for Nanopore amplicon data |  | https://github.com/Saskia-Oosterbroek/decona |
| TaxonTableTools | analyse and visualize their metabarcoding data quickly and reproducible via a graphical user interface | 10.1186/1471-2105-12-385 | https://pypi.org/project/taxontabletools/ |
| EasyMAP | user-friendly online platform for analyzing 16S ribosomal DNA sequencing data | 10.1016/j.nbt.2021.03.001 | http://easymap.cgm.ntu.edu.tw/ |
| Natrix  | open-source bioinformatics pipeline for the preprocessing of raw sequencing data | 10.1186/s12859-020-03852-4 | https://github.com/MW55/Natrix |
| LinDA | Linear Models for Differential Abundance Analysis of Microbiome Compositional Data | arXiv:2104.00242 | https://github.com/zhouhj1994/LinDA |
| Natrix-Pipeline | preprocessing of raw sequencing data | 10.1186/s12859-020-03852-4 | https://github.com/zelezniak-lab/Natrix |
| KAIJU | Fast and sensitive taxonomic classification for metagenomics | 10.1038/ncomms11257 | http://kaiju.binf.ku.dk/server |
| DAnIEL  | web server to analyze ITS amplicon sequencing data | 10.1101/2021.04.12.437814 | https://sbi.hki-jena.de/daniel/latest/ |
| MiDAS Field Guide | An ecosystem-specific reference database, taxonomy and knowledge platform for activated sludge and anaerobic digesters | 10.1016/j.watres.2020.115955 | https://www.midasfieldguide.org/guide/search |
| Calour | Interactive, Microbe-Centric Analysis Tool | 10.1128/mSystems.00269-18 | https://github.com/biocore/calour |
| microDecon | read‐subtraction tool for the post‐sequencing removal of contamination in metabarcoding studies | 10.1002/edn3.11 | https://github.com/donaldtmcknight/microDecon |
| JAMP | Just Another Metabarcoding Pipeline |  | https://github.com/VascoElbrecht/JAMP |
| chkMocks | Is the composition in experimental mock standards similar to theoretical expected composition |  | https://microsud.github.io/chkMocks/articles/aIntroduction.html |
| HCK and ABAA | Pipeline to Improve Fungi Metabarcoding Analysis | 10.3389/fmicb.2021.640693 | https://bitbucket.org/GottySG36/hck/src/master/ |
| SHOGUN  | profiles known taxonomic and gene abundances of short-read shotgun metagenomics sequencing data | 10.1093/bioinformatics/btaa277 | https://github.com/knights-lab/SHOGUN |
| MARES | pipeline and curated reference database for marine eukaryote metabarcoding | 10.1038/s41597-020-0549-9 | https://github.com/wpearman1996/MARES_database_pipeline |
| mTAGs | taxonomic profiling using degenerate consensus reference sequences of ribosomal RNA genes | 10.1093/bioinformatics/btab465 | https://github.com/SushiLab/mTAGs |
| MiCoNE  | Microbial Co-occurrence Network Explorer | 10.1128/msystems.00961-22 | https://github.com/segrelab/MiCoNE |
| Metage2Metabo | microbiota-scale metabolic complementarity for the identification of key species | 10.7554/eLife.61968 | https://github.com/AuReMe/metage2metabo , https://pypi.org/project/Metage2Metabo/ |
| MetaWRAP | a flexible pipeline for genome-resolved metagenomic data analysis | 10.1186/s40168-018-0541-1 | https://github.com/bxlab/metaWRAP |
| metaMATE | metabarcoding Multiple Abundance Threshold Evaluator | 10.1111/1755-0998.13337 | https://github.com/tjcreedy/metamate |
| Meta-SourceTracker | application of Bayesian source tracking to shotgun metagenomics | 10.7717/peerj.8783 | https://github.com/residentjordan/SourceTracker2-diagnostics |
| AlienTrimmer | quickly and accurately trim off multiple short contaminant sequences from high-throughput sequencing reads | 10.1016/j.ygeno.2013.07.011 | ftp://ftp.pasteur.fr/pub/gensoft/projects/AlienTrimmer/ |
| Blobology | exploring raw genome data for contaminants, symbionts, and parasites using taxon-annotated GC-coverage plots | 10.3389/fgene.2013.00237 | https://github.com/blaxterlab/blobology |
| Blobsplorer  | aids exploration and selection of subsets from TAGC-annotated data | 10.3389/fgene.2013.00237 | https://github.com/mojones/Blobsplorer |
| SourceTracker  | Bayesian community-wide culture-independent microbial source tracking | 10.1038/nmeth.1650 | https://sourceforge.net/projects/sourcetracker/ |
| MetagenoNets | inference as well as visualization of categorical, integrated (inter-omic) and bi-partite networks | 10.1093/nar/gkaa254 | https://web.rniapps.net/metagenonets |
| DnoisE | distance denoising by entropy | 10.7717/peerj.12758 | https://github.com/adriantich/DnoisE |
| Multiqc | A modular tool to aggregate results from bioinformatics analyses across many samples into a single report. | 10.1093/bioinformatics/btw354 | https://multiqc.info/ |
| CONTIGuator | contigs mapping over a reference genome | 10.1186/1751-0473-6-11 | http://contiguator.sourceforge.net |
| GapFiller | close gaps within scaffolds using paired reads | 10.1186/gb-2012-13-6-r56 | https://sourceforge.net/projects/gapfiller/ |
| Sealer | scalable gap-closing application for finishing draft genomes | 10.1186/s12859-015-0663-4 | https://github.com/bcgsc/abyss/tree/sealer-release |
| FGAP | automated gap closing tool | 10.1186/1756-0500-7-371 | https://github.com/pirovc/fgap |
| GAPPadder | closing gaps on draft genomes with short sequence reads | 10.1186/s12864-019-5703-4 | https://github.com/simoncchu/GAPPadder |
| PAGIT | post-assembly genome-improvement toolkit to obtain annotated genomes from contigs | 10.1038/nprot.2012.068 | https://www.sanger.ac.uk/tool/pagit/ |
| CloG | a pipeline for closing gaps in a draft assembly using short reads  | 10.1109/ICCABS.2011.5729881 |  |
| GapBlaster | Graphical Gap Filler for Prokaryote Genomes | 10.1371/journal.pone.0155327 | https://sourceforge.net/projects/gapblaster2015/ |
| PGcloser | Fast Parallel Gap-Closing Tool Using Long-Reads or Contigs to Fill Gaps in Genomes | 10.1177/1176934320913859 |  |
| DENTIST | using long reads for closing assembly gaps at high accuracy | 10.1093/gigascience/giab100 | https://github.com/a-ludi/dentist |
| TGS-GapCloser | fast and accurate gap closer for large genomes | 10.1093/gigascience/giaa094 | https://github.com/BGI-Qingdao/TGS-GapCloser |
| MetaPlatanus | metagenome assembler that combines long-range sequence links and species-specific features | 10.1093/nar/gkab831 | https://github.com/rkajitani/MetaPlatanus |
| METAnnotatorX2 | Comprehensive Tool for Deep and Shallow Metagenomic Data Set Analyses | 10.1128/mSystems.00583-21 | http://probiogenomics.unipr.it/cmu/ |
| nf-core/mag | best-practice pipeline for metagenome hybrid assembly and binning  | 10.1093/nargab/lqac007 | https://github.com/nf-core/mag |
| MUFFIN | Metagenomics workflow for hybrid assembly, differential coverage binning, metatranscriptomics and pathway analysis | 10.1371/journal.pcbi.1008716 | https://github.com/RVanDamme/MUFFIN |
| Lathe | A tool for generating bacterial genomes from metagenomes with Nanopore long read sequencing | 10.1038/s41596-020-00424-x | https://github.com/bhattlab/lathe |
| BusyBee  | metagenomic data analysis by bootstrapped supervised binning and annotation | 10.1093/nar/gkx348 | https://ccb-microbe.cs.uni-saarland.de/busybee |
| MaSuRCA  | genome assembler | 10.1093/bioinformatics/btt476 | https://github.com/alekseyzimin/masurca |
| MetaErg  | An Integrated Pipeline for Annotation and Visualization of Metagenomic Contigs | 10.3389/fgene.2019.00999 | https://github.com/xiaoli-dong/metaerg |
| MAGpy | downstream analysis of metagenome-assembled genomes (MAGs)  | 10.1093/bioinformatics/bty905 | https://github.com/WatsonLab/MAGpy |
| tidyMicro | a pipeline for microbiome data analysis and visualization using the tidyverse in R | 10.1186/s12859-021-03967-2 | https://github.com/CharlieCarpenter/tidyMicro |
| novelfams | Novel Microbial Gene Families | 10.1101/2022.01.26.477801 | https://novelfams.cgmlab.org/ |
| AGNOSTOS-DB | a resource to unlock the uncharted regions of the coding sequence space | 10.1101/2021.06.07.447314 | https://github.com/functional-dark-side/agnostos-wf |
| MATAM | Reconstruction of phylogenetic marker genes from short sequencing reads in metagenomes | 10.1093/bioinformatics/btx644 | https://github.com/bonsai-team/matam |
| marky-coco | pipeline based on de novo single-metagenome assembly for  characterization of hgc genes | 10.1101/2022.03.14.484253 | https://github.com/ericcapo/marky-coco |
| KEGGDecoder-binder | To calculate metabolic pathway completeness and build heatmaps from GhostKOALA output via KEGGDecoder |  | https://github.com/rotheconrad/KEGGDecoder-binder |
| KEGGDecoder | parse through KEGG-Koala outputs (blastKOALA, ghostKOALA, KOFAMSCAN) to determine completeness of metabolic pathways | 10.1038/s41396-018-0091-3 | https://github.com/bjtully/BioData/tree/master/KEGGDecoder |
| MicrobeAnnotator | Easy-to-use pipeline for the comprehensive metabolic annotation of microbial genomes. | 10.1186/s12859-020-03940-5 | https://github.com/cruizperez/MicrobeAnnotator |
| Functional Annotation |  |  | https://github.com/biovcnet/biovcnet.github.io/wiki/TOPIC:-Functional-Annotation |
| FastANI calculator |  |  | https://gtdb.ecogenomic.org/tools/fastani |
| MAG scripts | scripts used for MAGs for the Brazelton et al. manuscript on Lost City fluids MAGs |  | https://github.com/Brazelton-Lab/Brazelton-2022-Lost-City-fluids-MAGs/tree/main/scripts |
| IMP | Integrative Multi-species Prediction | 10.1093/nar/gkv486 | http://imp.princeton.edu/ |
| GraPhlAn | Compact graphical representation of phylogenetic data and metadata | 10.7717/peerj.1029 | http://segatalab.cibio.unitn.it/tools/graphlan/ |
| PropagAtE  | Prophage Activity Estimator: estimating if a prophage is in the lytic or lysogenic stage of infection | 10.1128/msystems.00084-22 | https://github.com/AnantharamanLab/PropagAtE |
| M2M - metage2metabo | microbiota-scale metabolic complementarity for the identification of key species | 10.7554/eLife.61968 | https://github.com/AuReMe/metage2metabo |
| eQuilibrator | a database solution for thermodynamic constant estimation | 10.1093/nar/gkab1106 | https://equilibrator.weizmann.ac.il/ |
| MicrobeAnnotator  | Easy-to-use pipeline for the comprehensive metabolic annotation of microbial genomes | 10.1186/s12859-020-03940-5 | https://github.com/cruizperez/MicrobeAnnotator |
| PhenDB  | Prediction of microbial phenotypes based on comparative genomics | 10.1186/1471-2105-16-S14-S1 | https://phendb.org/ |
| microeco | An R package for data mining in microbial community ecology | 10.1093/femsec/fiaa255 | https://github.com/ChiLiubio/microeco |
| Genomics Tutorial | Genomics Tutorial |  | https://genomics.sschmeier.com/introduction.html |
| RefSoil | A Reference Database for Genes and Traits of Soil Plasmids | 10.1128/mSystems.00349-18 | https://github.com/ShadeLab/RefSoil_plasmids |
| OGT_prediction | Predicting the optimal growth temperatures of prokaryotes using only genome derived features | 10.1093/bioinformatics/btz059 | https://github.com/DavidBSauer/OGT_prediction |
| Trait synthesis | A synthesis of bacterial and archaeal phenotypic trait data | 10.1038/s41597-020-0497-4 | https://github.com/bacteria-archaea-traits/bacteria-archaea-traits , https://github.com/bacteria-archaea-traits/bacteria-archaea-traits/releases/tag/v1.0.0 |
| Major dimensions | Cell size, genome size and maximum growth rate |  10.1002/ece3.7290  | https://github.com/bacteria-archaea-traits/major-dimensions , https://github.com/bacteria-archaea-traits/major-dimensions/releases/tag/v1.0.0 |
| TEMPURA | Database of Growth TEMPeratures of Usual and Rare Prokaryotes | 10.1264/jsme2.ME20074 | http://togodb.org/db/tempura |
| tRNA thermometer | using tRNA sequence to predict OGT | 10.1093/nar/gkaa1030 | https://bitbucket.org/bucklerlab/cnn_trna_ogt/ , https://doi.org/10.25739/a0g2-wb14 |
| KEMET | KEGG Module evaluation and microbial genome annotation expansion | 10.1016/j.csbj.2022.03.015 | https://github.com/Matteopaluh/KEMET |
| EukCC | estimation of eukaryotic genome quality in terms of completion and contamination, for the assessment of eukaryotic MAGs | 10.1186/s13059-020-02155-4 | https://github.com/Finn-Lab/EukCC |
| OrthoDB | hierarchical catalog of eukaryotic orthologs | 10.1093/nar/gkm845 | http://cegg.unige.ch/orthodb |
| PARSEC | localization and characterization of genomic sites in complete eukaryotic genomes |  |  |
| CulebrONT | long reads multi-assembler pipeline for prokaryotic and eukaryotic genomes | 10.1101/2021.07.19.452922 | https://pypi.org/project/culebrONT/ |
| Exogean | annotating protein-coding genes in eukaryotic genomic DNA | 10.1186/gb-2006-7-s1-s7 |  |
| QUAST-LG | genome assembly evaluation | 10.1093/bioinformatics/bty266 | http://cab.spbu.ru/software/quast-lg |
| Helixer | cross-species gene annotation of large eukaryotic genomes using deep learning | 10.1093/bioinformatics/btaa1044 | https://github.com/weberlab-hhu/Helixer |
| EukMetaSanity | prediction of eukaryote genomes | 10.1101/2021.07.25.453296 | https://github.com/cjneely10/EukMetaSanity |
| AMAW | automated gene annotation for nonmodel eukaryotic genomes | 10.1101/2021.12.07.471566 | https://bitbucket.org/phylogeno/amaw/ |
| BiSCoT | improving large eukaryotic genome assemblies with optical maps | 10.7717/peerj.10150 | http://www.genoscope.cns.fr/biscot  |
| diArk  | a resource for eukaryotic genome research | 10.1186/1471-2164-8-103 | http://www.diark.org/ |
| Tapestry | validate and edit small eukaryotic genome assemblies with long reads | 10.1101/2020.04.24.059402 | https://github.com/johnomics/tapestry |
| OPERA-LG | efficient and exact scaffolding of large, repeat-rich eukaryotic genomes  | 10.1186/s13059-016-0951-y | http://sourceforge.net/projects/operasf/ |
| OrthoFiller | utilising data from multiple species to improve the completeness of genome annotations | 10.1186/s12864-017-3771-x | https://github.com/mpdunne/orthofiller |
| Fluxer  | web application for the computation and interactive visualization of flux graphs from genome-scale metabolic models |  | https://fluxer.umbc.edu/ |
| ARGs-OAP | analysis pipeline for antibiotic resistance genes detection from metagenomic data | 10.1093/bioinformatics/btw136 | https://smile.hku.hk/SARGs |
| arg_ranker | detecting ARGs and assessing the ARG risks in metagenomes and genomes | 10.1038/s41467-021-25096-3 | https://github.com/caozhichongchong/arg_ranker |
| MICOM  | metabolic modeling of microbial communities  | 10.1128/mSystems.00606-19 | https://micom-dev.github.io/micom/ |
| COMETS | computer simulations of metabolism in spatially structured microbial communities | 10.1038/s41596-021-00593-3 | https://www.runcomets.org/about |
| Xander | employing a novel method for efficient gene-targeted metagenomic assembly | 10.1186/s40168-015-0093-6 | https://github.com/rdpstaff/Xander_assembler |
| MiDAS 4 | catalogue of full-length 16S rRNA gene sequences in wastewater treatment plants | 10.1038/s41467-022-29438-7 | https://www.midasfieldguide.org/guide |
| NMDC EDGE | metagenomic workflows  |  | https://nmdc-edge.org/home |
| RecruitPlotEasy | Advanced Read Recruitment Plot Tool for Assessing Metagenomic Population | 10.3389/fbinf.2021.826701 | https://github.com/KGerhardt/RecruitPlotEasy |
| antiSMASH 5.0 | secondary metabolite genome mining pipeline | 10.1093/nar/gkz310 | https://antismash.secondarymetabolites.org/#!/start |
| GapMind | Automated annotation of Amino acid biosynthesis | 10.1128/mSystems.00291-20 | https://papers.genomics.lbl.gov/cgi-bin/gapView.cgi |
| GapMind | Automated annotation of catabolism of small carbon sources | 10.1101/2021.11.02.466981 | https://papers.genomics.lbl.gov/cgi-bin/gapView.cgi?set=carbon |
| Microbiome Maps | Hilbert Curve Visualizations of Metagenomic Profiles | 10.1101/2021.03.22.436520 | https://www.microbiomemaps.org/ |
| Bactopia | complete analysis of bacterial genomes | 10.1128/mSystems.00190-20 | https://bactopia.github.io/v2.2.0/ |
| VirSorter2 | customizable pipeline to identify viral sequences from (meta)genomic data |  | https://bitbucket.org/MAVERICLab/virsorter2/src/master/ |
| namingGTDB |  | 10.20944/preprints202111.0557.v1 | https://github.com/quadram-institute-bioscience/namingGTDB |
| CoreGenomePrimers | Pangenome-guided primer design for clade-specific primer generation |  | https://github.com/leylabmpi/CoreGenomePrimers |
| MicRhoDE  | a curated database for the analysis of microbial rhodopsin diversity and evolution | 10.1093/database/bav080 | http://application.sb-roscoff.fr/micrhode/ |
| BacARscan | an in-silico resource to discern diversity in antibiotic resistance genes |  | https://github.com/University-of-Delhi-south-campus/BacARscan |
| MAGNETO | An Automated Workflow for Genome-Resolved Metagenomics | 10.1128/msystems.00432-22 | https://gitlab.univ-nantes.fr/bird_pipeline_registry/magneto |
| Tripal MapViewer | A tool for interactive visualization and comparison of genetic maps | 10.1093/database/baz100 | https://gitlab.com/mainlabwsu/tripal_map |
| BAC-BROWSER | Visualization and Analysis of Prokaryotic Genomes | 10.3389/fmicb.2018.02827 | http://smdb.rcpcm.org/tools/index.html |
| FLIM-MAP | Gene Context Based Identification of Functional Modules in Bacterial Metabolic Pathways | 10.3389/fmicb.2018.02183 | https://web.rniapps.net/FlimMap/ |
| AnnoTree | visualization and exploration of a functionally annotated microbial tree of life | 10.1093/nar/gkz246 | http://annotree.uwaterloo.ca/ |
| SLING | a tool to search for linked genes in bacterial datasets | 10.1093/nar/gky738 | https://github.com/ghoresh11/sling/wiki |
| Balrog | A universal protein model for prokaryotic gene prediction | 10.1371/journal.pcbi.1008727 | https://github.com/salzberg-lab/Balrog |
| PATRIC | Pathosystems Resource Integration Center | 10.1093/nar/gkz943 | https://patricbrc.org/ |
| coda4microbiome | Compositional Data Analysis for Microbiome Studies | 10.1101/2022.06.09.495511v1 | https://malucalle.github.io/coda4microbiome/ |
| SAMBA | Standardized and Automated MetaBarcoding Analyses workflow |  | https://github.com/ifremer-bioinformatics/samba |
| SHOOT | phylogenetic gene search and ortholog inference | 10.1186/s13059-022-02652-8 | https://shoot.bio/ |
| MarkerMAG | linking metagenome-assembled genomes (MAGs) with 16S rRNA marker genes using paired-end short reads | 10.1093/bioinformatics/btac398 | https://github.com/songweizhi/MarkerMAG |
| StrainXpress | strain aware metagenome assembly from short reads  | 10.1093/nar/gkac543 | https://github.com/HaploKit/StrainXpress |
| Pseudofinder | detection of pseudogenes in prokaryotic genomes | 10.1093/molbev/msac153 | https://github.com/filip-husnik/pseudofinder |
| getSequenceInfo | a suite of tools allowing to get genome sequence information from public repositories | 10.1186/s12859-022-04809-5 | https://github.com/karubiotools/getSequenceInfo |
| Simka | de novo comparative metagenomics tool | 10.7717/peerj-cs.94 | https://github.com/GATB/simka |
| SqueezeMeta | a fully automated metagenomics pipeline, from reads to bins | 10.3389/fmicb.2018.03349 | https://github.com/jtamames/SqueezeMeta |
| metaGT | improving and quantification metatranscriptome assembly using metagenome data |  | https://github.com/ablab/metaGT |
| METABOLIC | METabolic And BiogeOchemistry anaLyses In miCrobes | 10.1186/s40168-021-01213-8 | https://github.com/AnantharamanLab/METABOLIC |
| MetaPop | a pipeline for macro- and microdiversity analyses and visualization of microbial and viral metagenome-derived populations | 10.1186/s40168-022-01231-0 | https://github.com/metaGmetapop/metapop |
| Searching SRA | Similarity Searching the SRA |  | https://edwards.flinders.edu.au/similarity-searching-the-sra/ |
| SituSeq | remote and offline analysis of Nanopore-generated 16S rRNA amplicon data | 10.1038/s43705-023-00239-3 | https://github.com/jkzorz/SituSeq |
| PhiSpy | identifies prophages in Bacterial (and probably Archaeal) genomes | 10.1101/2023.04.20.537752 | https://github.com/linsalrob/PhiSpy |
| PHANOTATE | annotate phage genomes |  | https://github.com/linsalrob/PHANOTATE |
| FOCUS | An Agile Profiler for Metagenomic Data | 10.7717/peerj.425 | https://github.com/linsalrob/FOCUS |
| mapDamage2.0 | fast approximate Bayesian estimates of ancient DNA damage parameters  | 10.1093/bioinformatics/btt193 | https://github.com/ginolhac/mapDamage |
|PyDamage|automated ancient damage identification and estimation for contigs in ancient DNA de novo assembly|10.7717/peerj.11845|https://github.com/maxibor/pydamage|
| Fitness Browser | Browse thousands of genome-wide fitness experiments for diverse bacteria |  | https://fit.genomics.lbl.gov/ |
| differential-abundance analysis | taxonomic bias and differential-abundance analysis | 10.1101/2022.08.19.504330 | https://github.com/mikemc/differential-abundance-theory |
| taxumap | Visualize structure in large microbiome datasets |  | https://github.com/jsevo/taxumap |
| GOOS BioEco Portal | information on ocean biological and ecosystem observations |  | https://bioeco.goosocean.org/ |
| binny | an automated binning algorithm to recover high-quality genomes from complex metagenomic datasets | 10.1101/2021.12.22.473795 | https://github.com/a-h-b/binny |
| medna-metadata | open-source data management system for tracking environmental DNA samples and metadata | 10.1093/bioinformatics/btac556 | https://github.com/Maine-eDNA/medna-metadata |
| phyloMDA | phylogeny-aware microbiome data analysis | 10.1186/s12859-022-04744-5 | https://github.com/liudoubletian/phyloMDA |
| POST | phylogeny-guided microbiome OTU-specific association test | 10.1186/s40168-022-01266-3 | https://cran.r-project.org/web/packages/POSTm/index.html |
| PMG = Principal Microbial Groups | a novel approach that groups OTUs based only on relative abundances as an alternative to taxon grouping | 10.1093/bib/bbac328 | https://github.com/asliboyraz/PMGs |
| DeepToA | An Ensemble Deep-Learning Approach to Predicting the Theater of Activity of a Microbiome | 10.1093/bioinformatics/btac584 | https://plabase.informatik.uni-tuebingen.de/deeptoa/ |
| mOTUlizer | a robust Bayesian approach to leverage metagenome-assembled genomes for core-genome estimation | 10.1093/nargab/lqac060 | https://github.com/moritzbuck/mOTUlizer |
| SECOM  | Sparse Estimation of Correlations among Microbiomes for correlation analysis | 10.1038/s41467-022-32243-x | https://www.bioconductor.org/packages/release/bioc/html/ANCOMBC.html |
| APSCALE | advanced pipeline for simple yet comprehensive analyses of DNA Meta-barcoding data | 10.1093/bioinformatics/btac588 | https://github.com/DominikBuchner/apscale |
| MetaMeta  | integrating metagenome analysis tools to improve taxonomic profiling | 10.1186/s40168-017-0318-y | https://github.com/pirovc/metameta/ |
| PAIPline | Pipeline for the Automatic Identification of Pathogens |  | https://gitlab.com/andreas.andrusch/paipline |
| DaisySuite | pipeline for horizontal gene transfer (HGT) detection using sequencing data |  | https://gitlab.com/rki_bioinformatics/DaisySuite |
| gNOMO | a multi-omics pipeline for integrated host and microbiome analysis of non-model organisms |  | https://gitlab.com/rki_bioinformatics/gnomo |
| phytools | Phylogenetic Tools for Comparative Biology (and Other Things) |  | https://CRAN.R-project.org/package=phytools  |
| mixOmics | multivariate methods for the exploration and integration of biological datasets  | 10.1371/journal.pcbi.1005752 | http://mixomics.org/ |
| MixMC | Multivariate Statistical Framework to Gain Insight into Microbial Communities | 10.1371/journal.pone.0160169 | http://mixomics.org/mixMC/ |
| magi | Metabolites and Genes Integrated |  | https://github.com/biorack/magi |
| Simka | computes a large collection of standard ecological distances by replacing species counts by k-mer counts | 10.7717/peerj-cs.94 | https://gatb.inria.fr/software/simka/ |
| Commet | Comparing and combining multiple metagenomic datasets | 10.1109/BIBM.2014.6999135 | https://colibread.inria.fr/software/commet/ |
| sourmash | Quickly search, compare, and analyze genomic and metagenomic data sets. |  | https://github.com/sourmash-bio/sourmash |
| p4finder | identify P4-like elements in bacterial genome | 10.1098/rstb.2020.0475 | https://gitlab.pasteur.fr/jsousa/p4finder |
| MacSyFinder | Mine Genomes for Molecular Systems with an Application to CRISPR-Cas Systems | 10.1371/journal.pone.0110726 | https://github.com/gem-pasteur/macsyfinder |
| WIsH | Predicting prokaryotic hosts from metagenomic phage contigs | 10.1093/bioinformatics/btx383 | https://github.com/soedinglab/wish |
| PPR-Meta | identifying phages and plasmids from metagenomic fragments | 10.1093/gigascience/giz066 | https://github.com/zhenchengfang/PPR-Meta |
| Phage_Finder | Automated Identification of Prophage Regions in Complete Bacterial Genome Sequences | 10.1093/nar/gkl732 | http://phage-finder.sourceforge.net/ |
| iPHoP | host prediction for metagenome-assembled virus genomes | 10.1101/2022.07.28.501908 | https://bitbucket.org/srouxjgi/iphop |
| MetaPhage | Analyzing, Annotating, and Classifying Bacteriophages in Metagenomics Sequencing Data | 10.1128/msystems.00741-22 | https://github.com/MattiaPandolfoVR/MetaPhage |
| Evident | performing effect size and power calculations on microbiome data | 10.1101/2022.05.19.492684 | https://github.com/biocore/Evident |
| STATegra | Multi-Omics Data Integration | 10.3389/fgene.2021.620453 | https://www.bioconductor.org/packages/release/bioc/html/STATegRa.html |
| wiSDOM | R Shiny interface to conduct analysis and visualization of metagenomic data | 10.1093/bioinformatics/btab057 | https://github.com/lunching/wiSDOM |
| MOCHI | cross-platform tool for amplicon-based microbiota analysis | 10.1093/bioinformatics/btac494 | https://mochi.life.nctu.edu.tw/ |
| EzMAP | Easy Microbiome Analysis Platform | 10.1186/s12859-021-04106-7 | https://github.com/gnanibioinfo/EzMAP |
| iCAMP | Infer Community Assembly Mechanisms by Phylogenetic bin-based null model analysis | 10.1038/s41467-020-18560-z | https://github.com/DaliangNing/iCAMP1 |
| BiomeHorizon | Visualizing Microbiome Time Series Data in R | 10.1128/msystems.01380-21 | https://github.com/blekhmanlab/biomehorizon/ |
| mOTUs3 | estimates relative taxonomic abundance of microbial community members using metagenomic shotgun sequencing data | 10.1101/2021.04.20.440600 | https://github.com/motu-tool/mOTUs |
| ComplexHeatmap  | Complex heatmap visualization | 10.1002/imt2.43 | https://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html |
| mobr | Measurement of Biodiversity (MoB) | 10.1111/2041-210X.13102 | https://github.com/MoBiodiv/mobr |
| Meta-Apo | accuracy of 16Samplicon-based prediction of microbiome function | 10.1186/s12864-020-07307-1 | https://github.com/qibebt-bioinfo/meta-apo |
| PanACoTA | PANgenome with Annotations, COre identification, Tree and corresponding Alignments | 10.1093/nargab/lqaa106 | https://github.com/gem-pasteur/PanACoTA |
| HAYSTAC | robust and rapid species identification in high-throughput sequencing data | 10.1371/journal.pcbi.1010493 | https://github.com/antonisdim/HAYSTAC |
| TaxUMAP atlas | Visualize structure in large microbiome datasets | 10.1101/2022.09.27.509746 | https://github.com/jsevo/taxumap |
| DeepBIO  | biological sequence prediction, functional annotation, and visualization analysis | 10.1101/2022.09.29.509859 | http://inner.wei-group.net/DeepBIO |
| BusyBee Web | metagenomic data analysis by bootstrapped supervised binning and annotation | 10.1093/nar/gkx348 | https://ccb-microbe.cs.uni-saarland.de/busybee |
| gcMeta | a Global Catalogue of Metagenomics platform to support the archiving, standardization and analysis of microbiome data | 10.1093/nar/gky1008 | https://gcmeta.wdcm.org/ |
| ESPRIT | estimating species richness using large collections of 16S rRNA pyrosequences | 10.1093/nar/gkp285 | https://biotech.ufl.edu/people/sun/esprit.html |
| MoRS | Mortality by ribosomal sequencing (MoRS) provides a window into taxon-speciﬁc cell lysis | 10.1038/s41396-022-01327-3 | https://github.com/kevinzhongxu/tslysis |
| VEBA | recovery, clustering, and analysis of prokaryotic, microeukaryotic, and viral genomes from metagenomes | 10.1186/s12859-022-04973-8 | https://github.com/jolespin/veba |
| STRONG  | resolves strains on assembly graphs  |  | https://github.com/chrisquince/STRONG |
| biomonitoR | managing ecological data and calculating biomonitoring indices | 10.7717/peerj.14183 | https://github.com/alexology/biomonitoR |
| ngsLCA | fast and flexible lowest common ancestor inference and taxonomic profiling of metagenomic data | 10.1111/2041-210X.14006  | https://github.com/miwipe/ngsLCA |
| PHANTASM | PHylogenomic ANalyses for the TAxonomy and Systematics of Microbes | 10.1101/2022.10.18.512716 | https://github.com/dr-joe-wirth/phantasm |
| LotuS2 | ultrafast and highly accurate tool for amplicon sequencing analysis | 10.1186/s40168-022-01365-1 | http://lotus2.earlham.ac.uk/ |
| MIntO | Metagenomic and Metatranscriptomic Data Integration | 10.3389/fbinf.2022.846922 | https://github.com/arumugamlab/MIntO |
| StrainFacts  | Scalable Microbial Strain Inference in Metagenomic Data | 10.3389/fbinf.2022.867386 | https://github.com/bsmith89/StrainFacts |
| RecruitPlotEasy | Read Recruitment Plot Tool for Assessing Metagenomic Population Abundance and Genetic Diversity | 10.3389/fbinf.2021.826701 | https://github.com/KGerhardt/RecruitPlotEasy |
| Ptools | Pathway Tools Management of Pathway/Genome Data for Microbial Communities | 10.3389/fbinf.2022.869150 | https://biocyc.org/download.shtml |
| Graphia | graph-based approach for the visualisation and analysis of bacterial pangenomes | 10.1186/s12859-022-04898-2 | https://graphia.app/ |
| microTrait | Trait-Based Representation of Microbial Genomes | 10.3389/fbinf.2022.918853 | https://github.com/ukaraoz/microtrait |
| 16S-ITGDB | Integrated Database for Improving Species Classification of Prokaryotic 16S Ribosomal RNA Sequences | 10.3389/fbinf.2022.905489 | https://github.com/yphsieh/ItgDB |
| Metascan | METabolic Analysis, SCreening and ANnotation of Metagenomes | 10.3389/fbinf.2022.861505 | https://github.com/gcremers/metascan |
| GoldRush | de novo long read genome assembler with linear time complexity | 10.1101/2022.10.25.513734 | https://github.com/bcgsc/goldrush |
| CAMMiQ | Strain level microbial detection and quantification with applications to single cell metagenomics | 10.1038/s41467-022-33869-7 | https://github.com/algo-cancer/CAMMiQ |
| Bakta | rapid & standardized annotation of bacterial genomes, MAGs & plasmids | 10.1099/mgen.0.000685 | https://github.com/oschwengers/bakta |
| MAGScoT | bin-refinement tool | 10.1093/bioinformatics/btac694 | https://github.com/ikmb/MAGScoT |
| BRANEnet | embedding multilayer networks for omics data integration | 10.1186/s12859-022-04955-w | https://github.com/Surabhivj/BRANEnet |
| ResFinderFG v2.0 | database of antibiotic resistance genes obtained by functional metagenomics | 10.1101/2022.10.19.512667 | https://cge.food.dtu.dk/services/ResFinderFG/ |
| MetaNetX | Automated Model Construction and Genome Annotation for Large-Scale Metabolic Networks | 10.1093/nar/gkaa992 | https://www.metanetx.org/ |
| TaxonKit  | A Practical and Efficient NCBI Taxonomy Toolkit | 10.1016/j.jgg.2021.03.006 | https://github.com/shenwei356/taxonkit |
| CONSULT  | contamination removal from genomic sequencing reads | 10.1093/nargab/lqab071 | https://github.com/noraracht/CONSULT |
| EukCC | Estimating the quality of eukaryotic genomes recovered from metagenomic analysis | 10.1186/s13059-020-02155-4 | https://github.com/Finn-Lab/EukCC |
| CoverM | DNA read coverage and relative abundance calculator focused on metagenomics applications | 10.5281/zenodo.10531253 | https://github.com/wwood/CoverM |
| BusyBee  | convenient analysis of metagenomic data in the form of assembled contigs or long reads (PacBio or ONT) | 10.1093/nar/gkx348 | https://ccb-microbe.cs.uni-saarland.de/busybee |
| ProDeGe | computational protocol for fully automated decontamination of genomes | 10.1038/ismej.2015.100 |  |
| CheckM2 | rapid, scalable and accurate tool for assessing microbial genome quality | 10.1101/2022.07.11.499243 | https://github.com/chklovski/CheckM2 |
| MIDAS2 | Metagenomic Intra-species Diversity Analysis System | 10.1093/bioinformatics/btac713 | https://github.com/czbiohub/MIDAS2 |
| AsgeneDB | Arsenic metabolism gene database and computational tool for metagenome annotation | 10.1093/nargab/lqac080 | https://github.com/XinweiSong/Asgene |
| Phage Discovery Guide | Phage Discovery Guide |  | https://seaphagesphagediscoveryguide.helpdocsonline.com/home |
| KARGAMobile | Android app for portable, real-time, easily interpretable analysis of antibiotic resistance genes via nanopore sequencing | 10.3389/fbioe.2022.1016408 | https://github.com/Ruiz-HCI-Lab/KargaMobile |
| genoPlotR | comparative gene and genome visualization in R | 10.1093/bioinformatics/btq413 | https://genoplotr.r-forge.r-project.org/ |
| Squeegee | De novo identification of microbial contaminants in low microbial biomass microbiomes  | 10.1038/s41467-022-34409-z | https://gitlab.com/treangenlab/squeegee |
| Wochenende  | modular and flexible alignment-based shotgun metagenome analysis | 10.1186/s12864-022-08985-9 | https://github.com/MHH-RCUG/nf_wochenende |
| metaMIC | reference-free misassembly identification and correction of de novo metagenomic assemblies | 10.1186/s13059-022-02810-y | https://github.com/ZhaoXM-Lab/metaMIC |
| HOME  | HOst-Microbiota Evolution | 10.1111/1755-0998.13063 | https://github.com/BPerezLamarque/HOME |
| RPANDA | R: Phylogenetic ANalyses of DiversificAtion | 10.1111/2041-210X.12526 | https://github.com/hmorlon/PANDA |
| RPANDA | measure the phylogenetic signal in a bipartite interaction network | 10.1101/2021.08.30.458192 | https://github.com/BPerezLamarque/Phylosignal_network |
| ALE | Reconstruction of reconciled gene trees and estimation of DTL rates from a sample of gene trees and a rooted species tree | 10.1093/sysbio/syt054 | https://github.com/ssolo/ALE/ |
| vertical transmission | Scripts for running methods to detect vertical transmission | 10.1111/1755-0998.13063 | https://github.com/BPerezLamarque/Scripts/tree/master/Comparing_methods_vertical_transmission |
| MEGARes and AMR++ | comprehensive database of antimicrobial resistance determinants and an improved software pipeline for classification | 10.1093/nar/gkac1047 | https://www.meglab.org/ |
| metaMIC | reference-free misassembly identification and correction of de novo metagenomic assemblies | 10.1186/s13059-022-02810-y | https://github.com/ZhaoXM-Lab/metaMIC |
| MetaGT | de novo assembly of metatranscriptomes with the aid of metagenomic data | 10.3389/fmicb.2022.981458 | https://github.com/ablab/metaGT |
| multi-metagenome | Metagenome assembly guide | 10.1038/nbt.2579 | https://github.com/MadsAlbertsen/multi-metagenome |
| mmgenome | extracting individual genomes from metagenomes | 10.1101/059121 | http://madsalbertsen.github.io/mmgenome/ |
| ampvis2  | visualising amplicon data | 10.1101/299537 | https://github.com/KasperSkytte/ampvis2 |
| MAGE (Microbial Abundance GaugE) | Strain Level Profiling of Metagenome Samples | 10.1101/2022.11.24.517382| | 
|MaGe (Magnifying Genomes) |microbial genome annotation system supported by synteny results|10.1093/nar/gkj406|https://mage.genoscope.cns.fr/microscope/home/index.php|
|mi-atlas|interactive and evolving catalog of microbial interactions|10.1093/femsle/fnz125|https://cpauvert.github.io/mi-atlas/|
|Namco|microbiome explorer|10.1099/mgen.0.000852|https://exbio.wzw.tum.de/namco/|
|PathSeq|discovery and identification of microbial sequences in genomic and transcriptomic libraries from eukaryotic hosts|10.1093/bioinformatics/bty501|https://github.com/broadinstitute/gatk|
|VIRify|detection, annotation and taxonomic classification pipeline of viruses|10.1371/journal.pcbi.1011422|https://github.com/EBI-Metagenomics/emg-viral-pipeline|
|proGenomes3|accurately and consistently annotated high-quality prokaryotic genomes|10.1093/nar/gkac1078|https://progenomes.embl.de/|
|Systems biology slides|||https://gitlab.com/principlescellphysiology/teaching-material/-/tree/master/systems_biology|
|metacoder|parsing, plotting, and manipulating large taxonomic data sets||https://grunwaldlab.github.io/metacoder_documentation/index.html|
|phyloseq Extensions Index|||http://joey711.github.io/phyloseq-extensions/extensions-index.html|
|The R Graph Gallery|||https://r-graph-gallery.com/index.html|
|RdRp-scan|identify and annotate divergent RNA viruses in metagenomic sequence data|10.1093/ve/veac082|https://github.com/JustineCharon/RdRp-scan/|
|MetaPredict|predicting metabolic modules in incomplete bacterial genomes|10.1101/2022.12.21.521254|https://github.com/d-mcgrath/MetaPredict|
|Greengenes2|Greengenes2 phylogeny|10.1101/2022.12.19.520774|https://github.com/biocore/greengenes2|
|PathIN|visualization of pathway interaction networks|10.1016/j.csbj.2022.12.028|https://pathin.cing-big.hpcf.cyi.ac.cy/|
|FlashWeave|predict ecological interactions between microbes from large-scale compositional abundance data|10.1016/j.cels.2019.08.002|https://github.com/meringlab/FlashWeave.jl|
|Victors|a web-based knowledge base of virulence factors in human and animal pathogens|10.1093/nar/gky999|http://www.phidias.us/victors|
|KEMET|python tool for KEGG Module evaluation and microbial genome annotation expansion|10.1016/j.csbj.2022.03.015|https://github.com/Matteopaluh/KEMET|
|MeRRCI|Metagenome, Resistome, and Replicome for Causal Inferencing|10.1099/mgen.0.000899|https://github.com/stebliankin/merrci|
|Bacterial Genetics Course Packet|Course packets for Bacterial Genetics and Physiology||https://github.com/graymicrolab/Bacterial_Genetics_Course_Packet|
|Tools for Microbiome Analysis|A list of R environment based tools for microbiome data exploration, statistical analysis and visualization||https://microsud.github.io/Tools-Microbiome-Analysis/|
|SprayNPray|user-friendly taxonomic profiling of genome and metagenome contigs|10.1186/s12864-022-08382-2|https://github.com/Arkadiy-Garber/SprayNPray|
|OpenGenomeBrowser|web platform for genome data management and comparative genomics|10.1186/s12864-022-09086-3|https://opengenomebrowser.github.io/|
|MicNet toolbox|Visualizing and unraveling a microbial network|10.1371/journal.pone.0259756|https://github.com/Labevo/MicNetToolbox|
|MetaBinner|binning method to recover individual genomes from complex microbial communities|10.1186/s13059-022-02832-6|https://github.com/ziyewang/MetaBinner|
|gget|Efficient querying of genomic reference databases|10.1093/bioinformatics/btac836|https://github.com/pachterlab/gget|
|FastViromeExplorer-Novel|Recovering Draft Genomes of Novel Viruses and Phages in Metagenomic Data|10.1089/cmb.2022.0397|https://github.com/saima-tithi/FVE-novel|
|PhyloPlus|Phylogenetic Interrogation of Metagenomic Communities|10.1128/mbio.03455-22|https://github.com/Dennis-xyHuang/PhyloPlus|
|skani|Fast and robust metagenomic sequence comparison through sparse chaining|10.1101/2023.01.18.524587|https://github.com/bluenote-1577/skani|
|EasyAmplicon|pipeline for amplicon data analysis|10.1002/imt2.83|https://github.com/YongxinLiu/EasyAmplicon|
|PhyloPic|Free silhouette images of animals, plants, and other life forms||http://phylopic.org/|
|VTAM|pipeline for validating metabarcoding data using controls|10.1016/j.csbj.2023.01.034|https://github.com/aitgon/vtam|
|ViWrap|identify, bin, classify, and predict viral-host relationships for viruses from metagenomes|10.1101/2023.01.30.526317|https://github.com/AnantharamanLab/ViWrap|
|BIRDMAn|robust inference of host-microbe associations|10.1101/2023.01.30.526328|https://github.com/biocore/BIRDMAn|
|Snippy|Rapid haploid variant calling and core genome alignment||https://github.com/tseemann/snippy|
|REALPHY|Reference sequence Alignment based Phylogeny builder|10.1093/molbev/msu088|https://realphy.unibas.ch/realphy/|
|stRainy|assembly-based metagenomic strain phasing using long reads|10.1038/s41592-024-02424-1|https://github.com/katerinakazantseva/stRainy|
|FuncDiv| computation of contributional diversity metrics from microbiome data|10.1093/bioinformatics/btac809|https://github.com/gavinmdouglas/FuncDiv|
|InStrain|analysis of co-occurring genome populations from metagenomes|10.1038/s41587-020-00797-0|https://github.com/MrOlm/instrain|
|gcplyr|import, wrangle, and do model-free analyses of microbial growth curve data||https://mikeblazanin.github.io/gcplyr/|
|Symbiont-screener|reference-free tool to separate host sequences from symbionts for error-prone long reads|10.3389/fmars.2023.1087447|https://github.com/BGI-Qingdao/Symbiont-Screener|
|DeepBIO|platform for high-throughput biological sequence prediction, functional annotation and visualization analysis|10.1093/nar/gkad055|https://inner.wei-group.net/DeepBIO/|
|gutSMASH|predict specialized primary metabolic pathways from the human gut microbiota|10.1038/s41587-023-01675-1|https://github.com/victoriapascal/gutsmash/tree/gutsmash|
|VirBot|RNA viral contig detector for metagenomic data|10.1093/bioinformatics/btad093|https://github.com/GreyGuoweiChen/RNA_virus_detector|
|BinaRena|dedicated interactive platform for human-guided exploration and binning of metagenomes|10.1101/2022.06.28.498025|https://github.com/qiyunlab/binarena|
|StrainSelect|novel microbiome reference database that disambiguates all bacterial strains, genome assemblies and extant cultures worldwide|10.1016/j.heliyon.2023.e13314|http://strainselect.secondgenome.com|
|MGnify Genomes|biome-specific microbial genome catalogues|10.1016/j.jmb.2023.168016|https://www.ebi.ac.uk/metagenomics/browse/genomes/|
|kb_DRAM|Annotating and functional profiling of genomes with DRAM in KBase|10.1093/bioinformatics/btad110|https://github.com/shafferm/kb_DRAM|
|GT-Pro|ultra-rapid genotyping of prokaryotes in shotgun metagenomes|10.1038/s41587-021-01102-3|https://github.com/zjshi/gt-pro|
|Maast|microbial agile accurate SNP typing|10.1101/2022.07.06.499075|https://github.com/zjshi/Maast|
|Bactabolize|high-throughput genome-scale metabolic reconstruction and growth simulation pipeline|10.1101/2023.02.26.530115|https://github.com/kelwyres/Bactabolize|
|MetaPhlAn 4|Metagenomic Phylogenetic Analysis|10.1038/s41587-023-01688-w|https://github.com/biobakery/MetaPhlAn|
|Bacteriophages playing nice|gut bacteria-phage interactions|10.1016/j.isci.2023.106007|https://github.com/sgsutcliffe/Bacteriophages_Playing_Nice|
|BlobToolKit|Interactive Quality Assessment of Genome Assemblies|10.1534/g3.119.400908|https://blobtoolkit.genomehubs.org/|
|perfect bacterial genome |Assembling the perfect bacterial genome using Oxford Nanopore and Illumina sequencing |10.1371/journal.pcbi.1010905|https://github.com/rrwick/Perfect-bacterial-genome-tutorial|
|inStrain |profiles population microdiversity from metagenomic data and sensitively detects shared microbial strains|10.1038/s41587-020-00797-0|https://github.com/MrOlm/inStrain|
|fastq-dl|Download FASTQ files from the European Nucleotide Archive or the Sequence Read Archive repositories||https://github.com/rpetit3/fastq-dl|
|SatelliteFinder|Detection of four families of phage satellites: P4-like, PICI, cfPICI and PLE in protein datasets|10.1093/nar/gkad123|https://hub.docker.com/r/gempasteur/satellite_finder|
|SnakeMAGs|workflow to reconstruct prokaryotic genomes from metagenomes|10.12688/f1000research.128091.1|https://github.com/Nachida08/SnakeMAGs|
|geNomad|classification and annotation framework to identify sequences of plasmids and viruses|10.1101/2023.03.05.531206|https://portal.nersc.gov/genomad|
|dragonflye|Assemble bacterial isolate genomes from Nanopore reads||https://github.com/rpetit3/dragonflye|
|hybracter|Hybrid (and long-only) bacterial assembly pipeline|10.1099/mgen.0.001244|https://github.com/gbouras13/hybracter|
|Trycycler|generating consensus long-read assemblies for bacterial genomes|10.1186/s13059-021-02483-z|https://github.com/rrwick/Trycycler|
|plassembler|Automated Bacterial Plasmid Assembly Program|10.1093/bioinformatics/btad409|https://github.com/gbouras13/plassembler|
|PhaGAA|integrated web server platform for phage genome annotation and analysis|10.1093/bioinformatics/btad120|phage.xialab.info|
|eOmics|an R package for improved omics data analysis|10.1101/2023.03.11.532240|https://github.com/yuabrahamliu/eOmics|
|compare_genomes|a comparative genomics workflow to streamline the analysis of evolutionary divergence across genomes|10.1101/2023.03.16.533049|https://github.com/jeffersonfparil/compare_genomes|
|SCRuB|Source tracking for Contamination Removal in microBiomes|10.1038/s41587-023-01696-w|https://github.com/Shenhav-and-Korem-labs/SCRuB|
|demografr| Approximate Bayesian Computation (ABC) toolkit for R||https://github.com/bodkan/demografr|
|MetaErg|Metagenome Annotation Pipeline||https://xiaoli-dong.github.io/metaerg/|
|plastiC|pipeline for recovery and characterization of plastid genomes from metagenomic datasets|10.12688/wellcomeopenres.19589.1|https://github.com/Finn-Lab/plastiC|
|r-statistics co|educational resource for computing in R||http://r-statistics.co/|
|Phenonaut|toolkit for multi-omic phenotypic space exploration|10.1093/bioinformatics/btad143|https://github.com/CarragherLab/phenonaut|
|KARGVA|K-mer antibiotic resistance gene variant analyzer|10.3389/fmicb.2023.1060891|https://github.com/DataIntellSystLab/KARGVA|
|KARGA|K-mer-based Antibiotic Resistance Gene (ARG) Analysis of High-Throughput Sequencing Data|10.1109/BHI50953.2021.9508479|https://github.com/DataIntellSystLab/KARGA|
|Secmarker|Computational identification of the selenocysteine tRNA (tRNASec) in genomes|10.1371/journal.pcbi.1005383|https://secmarker.crg.es/|
|MicrobeCensus|estimate average genome size from shotgun metagenomic data|10.1186/s13059-015-0611-7|https://github.com/snayfach/MicrobeCensus|
|metasssnake2|snakemake/python based metagenomics pipeline||https://github.com/moritzbuck/0053_metasssnake2|
|MetAnnotate|function-specific taxonomic profiling and comparison of metagenomes|10.1186/s12915-015-0195-4|https://github.com/MetAnnotate/MetAnnotate|
|UProC|tools for ultra-fast protein domain classification|10.1093/bioinformatics/btu843|https://github.com/gobics/uproc|
|mi-faser|microbiome - functional annotation of sequencing reads|10.1093/nar/gkx1209|https://bitbucket.org/bromberglab/mifaser/src/master/|
|fusionDB|assessing microbial diversity and environmental preferences via functional similarity networks|10.1093/nar/gkx1060|https://services.bromberglab.org/fusiondb/|
|fusion |functional-repertoire similarity-based organism network|10.1371/journal.pcbi.1004472|https://bromberglab.org/?q=services|
|Predict-Protein|predicting secondary structure and returning families of related proteins||https://bitbucket.org/bromberglab/predictprotein/src/master/|
|fuNTRp|creating functional maps of protein sequences||https://bitbucket.org/bromberglab/funtrp/src/master/|
|pEffect|predicts bacterial type III effector proteins||https://services.bromberglab.org/peffect/|
|Herodotools|An R package to integrate macroevolution, biogeography and community ecology|10.1111/jbi.14610|https://github.com/GabrielNakamura/Herodotools|
|Ensemble Quotient Optimization (mEQO) for microbiome|Annotation-free discovery of functional groups in microbial communities|10.1038/s41559-023-02021-z|https://github.com/Xiaoyu2425/mEQO|
|ViroProfiler|viral metagenomic data analysis|10.1080/19490976.2023.2192522|https://github.com/deng-lab/viroprofiler|
|PanKmer|k-mer based and reference-free pangenome analysis|10.1101/2023.03.31.535143|https://gitlab.com/salk-tm/pankmer|
|PanGraphViewer|A Versatile Tool to Visualize Pangenome Graphs|10.1101/2023.03.30.534931|https://github.com/TF-Chan-Lab/panGraphViewer|
|UMAP|Uniform Manifold Approximation and Projection for Dimension Reduction|10.48550/arXiv.1802.03426|https://github.com/lmcinnes/umap|
|AMON|annotation of metabolite origins via networks to integrate microbiome and metabolome data|10.1186/s12859-019-3176-8|https://github.com/lozuponelab/AMON|
|GenoVi|automated circular genome visualizer for bacteria and archaea|10.1371/journal.pcbi.1010998|https://github.com/robotoD/GenoVi|
|Phables|from fragmented assemblies to high-quality bacteriophage genomes|10.1093/bioinformatics/btad586|https://github.com/Vini2/phables|
|PanGenome Graph Builder (PGGB)|Building pangenome graphs|10.1101/2023.04.05.535718|https://github.com/pangenome/pggb|
|ClassGraph|Improving Metagenomic Read Classification with Overlap Graphs|10.1089/cmb.2022.0208|https://github.com/MargheritaCavattoni/ClassGraph|
|SUPER-FOCUS|agile functional analysis of metagenomic data|10.1093/bioinformatics/btv584|https://github.com/metageni/SUPER-FOCUS|
|PLASS |Protein-Level ASSembler||https://github.com/soedinglab/plass|
|MGX|management and analysis of metagenomic datasets|10.1186/s40168-018-0460-1|https://mgx-metagenomics.github.io/|
|MGX 2.0|Shotgun- and assembly-based metagenome and metatranscriptome analysis from a single source|10.1101/2023.09.21.558800|https://mgx-metagenomics.github.io|
|LookingGlass |deep learning model for read-length biological sequences|10.1038/s41467-022-30070-8|https://github.com/ahoarfrost/LookingGlass/|
|fastBio |manipulating data and creating and training deep learning models for biological sequencing data|10.1038/s41467-022-30070-8|https://github.com/ahoarfrost/fastBio/|
|ShortBRED|High-Specificity Targeted Functional Profiling in Microbial Communities|10.1371/journal.pcbi.1004557|https://huttenhower.sph.harvard.edu/shortbred/|
|transXpress|streamlined de novo transcriptome assembly and annotation|10.1186/s12859-023-05254-8|https://github.com/transXpress/transXpress|
|microbial-genes| gene catalog of 17 different microbial ecosystems|10.1128/msystems.00118-23|https://www.microbial-genes.bio/|
|MJOLNIR|Metabarcoding Joining Obitools & Linkage Networks In R||https://github.com/uit-metabarcoding/MJOLNIR|
|MJOLNIR3|Metabarcoding Joining Obitools & Linkage Networks In R||https://github.com/adriantich/MJOLNIR3|
|EvoMIL|Prediction of virus-host association |10.1101/2023.04.07.536023|https://github.com/liudan111/EvoMIL|
|CyanoSeq|a database of cyanobacterial 16S rRNA sequences with curated taxonomy|10.1111/jpy.13335|https://zenodo.org/record/7569105|
|GALBA|Genome Annotation with Miniprot and AUGUSTUS|10.1101/2023.04.10.536199|https://github.com/Gaius-Augustus/GALBA|
|THAPBI PICT|fast, cautious, and accurate metabarcoding analysis pipeline|10.1101/2023.03.24.534090|https://github.com/peterjc/thapbi-pict/|
|AmphoraNet|webserver implementation of the AMPHORA2 metagenomic workflow suite|10.1016/j.gene.2013.10.015|http://amphoranet.pitgroup.org|
|AMPHORA2|An Automated Phylogenomic Inference Pipeline for Bacterial and Archaeal Sequences|10.1093/bioinformatics/bts079|https://github.com/wu-lab-uva/AMPHORA2|
|Phyla_AMPHORA|A Phylum-specific Automated Phylogenomic Inference Pipeline for Bacterial Sequences|10.1093/molbev/mst059|https://github.com/wu-lab-uva/Phyla_AMPHORA|
|YAMP|containerized workflow enabling reproducibility in metagenomics research|10.1093/gigascience/giy072|https://github.com/alesssia/YAMP|
|Metagenome-Atlas|a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data|10.1186/s12859-020-03585-4|https://github.com/metagenome-atlas/atlas|
|cogeqc|assess genome assembly and annotation quality, orthogroup inference, and synteny detection|10.1101/2023.04.14.536860|https://github.com/almeidasilvaf/cogeqc|
|UGMAGrefiner|increase completeness and resolution of metagenome-assembled genomes|10.1016/j.csbj.2023.03.030||
|BarcoSim |simulates DNA barcode sequences||https://github.com/loukesio/barcosim|
|Making maps with R|||https://www.paulamoraga.com/tutorial-maps/|
|CORRAL |identification of eukaryotes in shotgun metagenomic data|10.1186/s40168-023-01505-1|github.com/wbazant/CORRAL|
|cblindplot|create colour blind friendly plots|10.1016/j.ecoinf.2023.102045|https://github.com/ducciorocchini/cblindplot|
|PopCOGenT|Identifying microbial populations using networks of horizontal gene transfer|10.1016/j.cell.2019.06.033|https://github.com/philarevalo/PopCOGenT|
|SAVEMONEY|Barcode-free multiplex plasmid sequencing using Bayesian analysis and nanopore sequencing|10.1101/2023.04.12.536413|https://github.com/MasaakiU/MultiplexNanopore|
|microshades |custom color shading palettes that improve accessibility and data organization||https://github.com/KarstensLab/microshades|
|YACHT|an ANI-based statistical test to detect microbial presence/absence in a metagenomic sample|10.1101/2023.04.18.537298|https://github.com/KoslickiLab/YACHT|
|LucaProt|A novel deep learning framework that incorporates protein amino acid sequence and structure information to predict protein function|10.1101/2023.04.18.537342|https://github.com/alibaba/LucaProt|
|rtrees|an R package to assemble phylogenetic trees from megatrees|10.1111/ecog.06643|https://github.com/daijiang/rtrees|
|U.PhyloMaker|An R package that can generate large phylogenetic trees for plants and animals|10.1016/j.pld.2022.12.007|https://github.com/jinyizju/U.PhyloMaker|
|HOTSPOT|Hierarchical hOst predicTion for aSsembled Plasmid cOntigs with Transformer|10.1093/bioinformatics/btad283|https://github.com/Orin-beep/HOTSPOT|
|Recentrifuge|Robust comparative analysis and contamination removal for metagenomics|10.1371/journal.pcbi.1006967|https://github.com/khyox/recentrifuge|
|hkgfinder|Find housekeeping genes in prokaryotic (meta)genomes||https://github.com/Ebedthan/hkgfinder|
|Conditional Quantile Regression (ConQuR)|a comprehensive microbiome batch effects removal tool|10.1038/s41467-022-33071-9|https://github.com/wdl2459/ConQuR|
|ggpicrust2 |an R package for PICRUSt2 predicted functional profile analysis and visualization|arXiv:2303.10388|https://github.com/cafferychen777/ggpicrust2|
|Read2Tree|Inference of phylogenetic trees directly from raw sequencing reads |10.1038/s41587-023-01753-4|https://github.com/DessimozLab/read2tree|
|Calculation of the Percentage Of Conserved Proteins (POCP)|proposed genus boundary for the prokaryotes based on genomic insights|10.1128/JB.01688-14|https://github.com/hoelzer/pocp|
|MetQuest |identifying all possible pathways from metabolic networks between the source and the target metabolites|10.1038/s41598-018-28007-7|https://github.com/RamanLab/metquest2|
|Residual neural network for Misassembled Contig identification (ResMiCo)|Increasing the quality of metagenome-assembled genomes with deep learning|10.1371/journal.pcbi.1011001|https://github.com/leylabmpi/ResMiCo|
|KOunt | calculating KEGG orthologue abundance in metagenomic sequence data|10.1101/2023.04.27.538265|https://github.com/WatsonLab/KOunt|
|TOGA|Tool to infer Orthologs from Genome Alignments|10.1126/science.abn3107|https://github.com/hillerlab/TOGA|
|GUNC|detection of chimerism and contamination in prokaryotic genomes|10.1186/s13059-021-02393-0|https://github.com/grp-bork/gunc|
|SoilTemp|A global database of near-surface temperature|10.1111/gcb.15123|https://www.soiltempproject.com/|
|BacterAI|automated science platform that maps microbial metabolism but requires no prior knowledge|10.1038/s41564-023-01376-0|https://github.com/jensenlab/BacterAI|
|Proksee|in-depth characterization and visualization of bacterial genomes|10.1093/nar/gkad326|https://proksee.ca/|
|Genomes on a Tree (GoaT)|search engine for genomic and sequencing project metadata across the eukaryotic tree of life|10.12688/wellcomeopenres.18658.1|https://goat.genomehubs.org/|
|pvogs_function|predicting functional associations between pVOGs|10.1186/s12859-021-04343-w|https://github.com/MGXlab/pvogs_function|
|phap |Phage Host Analysis Pipeline||https://github.com/MGXlab/phap|
|CAT and BAT|Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT)|10.1186/s13059-019-1817-x|https://github.com/MGXlab/CAT|
|Social Niche Breadth (SNB)|A social niche breadth score reveals niche range strategies of generalists and specialists|10.1038/s41559-023-02027-7|https://github.com/MGXlab/social_niche_breadth_SNB|
|nf-core/eager|best-practise processing pipeline for genomic NGS sequencing data, with a focus on ancient DNA (aDNA) data|10.7717/peerj.10947|https://github.com/nf-core/eager|
|PHASTEST|PHAge Search Tool with Enhanced Sequence Translation|10.1093/nar/gkad382|https://phastest.ca/|
|clusterProfiler|enrichment tool for interpreting omics data|10.1016/j.xinn.2021.100141|https://github.com/YuLab-SMU/clusterProfiler|
|DePolymerase Predictor (DePP)|targeted identification of phage depolymerases|10.1186/s12859-023-05341-w|https://github.com/DamianJM/Depolymerase-Predictor|
|GIL| python package for designing custom indexing primers |10.1093/bioinformatics/btad328|https://github.com/de-Boer-Lab/GIL|
|VirPipe|pipeline for detecting viral genomes from Nanopore sequencing|10.1093/bioinformatics/btad293|https://github.com/KijinKims/VirPipe|
|WebQUAST|online evaluation of genome assemblies |10.1093/nar/gkad406|https://www.ccb.uni-saarland.de/quast|
|gRodon|an R package to estimate maximal growth rates of prokaryotes|10.1073/pnas.2016810118|https://github.com/jlw-ecoevo/gRodon2|
|gRodon2|an R package to estimate maximal growth rates of microbial eukaryotes|10.1101/2021.10.15.464604|https://github.com/jlw-ecoevo/gRodon2|
|Metagenome Mode v2|Community-Wide Estimates of Growth Potential from Metagenomes|10.1128/msystems.00745-22|https://github.com/jlw-ecoevo/gRodon2|
|Growth Rate InDex (GRiD)|estimating in situ growth rates of ultra-low coverage (>0.2×) and de novo-assembled metagenomes|10.1038/s41467-018-07240-8|https://github.com/ohlab/GRiD|
|Strain-level Metagenomic Estimation of Growth rate (SMEG)|measures growth rates of bacterial subspecies or strains from complex metagenomic samples|10.1126/sciadv.aaz2299|https://github.com/ohlab/SMEG|
|Pest Alert Tool|a web-based application for flagging species of concern in metabarcoding datasets|10.1093/nar/gkad364|https://pest-alert-tool-prod.azurewebsites.net/|
|CRABS|Creating Reference databases for Amplicon-Based Sequencing |10.1111/1755-0998.13741|https://github.com/gjeunen/reference_database_creator|
|itol.toolkit| working with iTOL (Interactive Tree Of Life) by an automated generation of annotation files|10.1093/bioinformatics/btad339|https://github.com/TongZhou2017/itol.toolkit|
|Genekitr|R package and a corresponding web server to assist exploring large gene sets|10.1186/s12859-023-05342-9|https://github.com/GangLiLab/genekitr|
|microbiomeExplorer|analyze and visualize microbial community sequencing data||https://github.com/zoecastillo/microbiomeExplorer|
|Anansi|Knowledge-based Integration of Multi-Omic Datasets|10.48550/arXiv.2305.10832|https://github.com/thomazbastiaanssen/anansi|
|Pygenomics|manipulating genomic intervals and data files in Python|10.1093/bioinformatics/btad346|https://gitlab.com/gtamazian/pygenomics|
|Phanta|Rapidly quantify taxa from all domains of life, directly from short-read human gut metagenomes|10.1038/s41587-023-01799-4|https://github.com/bhattlab/phanta|
|Minigraph-Cactus|combine multiple reference genome assemblies into a pangenome reference graph|10.1038/s41587-023-01828-2|https://github.com/ComparativeGenomicsToolkit/cactus/blob/master/doc/pangenome.md|
|MgCod|Gene Prediction in Phage Genomes with Multiple Genetic Codes|10.1016/j.jmb.2023.168159|https://github.com/gatech-genemark/MgCod|
|miaSim|an R/Bioconductor package to easily simulate microbial community dynamics|10.1111/2041-210X.14129|https://github.com/microbiome/miaSim|
|FALCON |analysis of nestedness in bipartite network|10.12688/f1000research.4831.1|https://github.com/sjbeckett/FALCON|
|Arcadia-Science/metagenomics|pipeline for profiling metagenomes obtained through either Illumina or Nanopore technologies||https://github.com/Arcadia-Science/metagenomics|
|Py2R |code translator tool from Python to R or from R to Python||https://github.com/ianandersonlol/Py2R|
|DAS Tool|calculate an optimized, non-redundant set of bins from a single assembly|10.1038/s41564-018-0171-1|https://github.com/cmks/DAS_Tool|
|Bacterial Wars (BW)|prediction of bacterial predominance based on network analysis measures|10.1093/nargab/lqad049|https://bioinformatics.cing.ac.cy/bacterialwars|
|Contig Overlap Based Re-Assembly (COBRA)|get higher quality virus genomes assembled from short-read metagenomes|10.1101/2023.05.30.542503|https://github.com/linxingchen/cobra.github.io|
|rCRUX|Generating Metabarcoding Reference libraries in R|10.1101/2023.05.31.543005|https://github.com/CalCOFI/rCRUX|
|Metabuli|metagenomic classifier that jointly analyze both DNA and amino acid (AA) sequences|10.1101/2023.05.31.543018|https://github.com/steineggerlab/Metabuli|
|iTaxoTools|specimen-based software toolkit for taxonomists|10.11646/MEGATAXA.6.2.1|https://github.com/iTaxoTools|
|U-CIE|Color encoding of high-dimensional data|10.1002/pro.4388|https://github.com/mikelkou/ucie|
|kmindex|indexing and querying sequencing samples||https://github.com/tlemane/kmindex|
|miniBUSCO|a faster and more accurate reimplementation of BUSCO|10.1101/2023.06.03.543588|https://github.com/huangnengCSU/minibusco|
|CViewer|integration of shotgun metagenomics with other omics datasets|10.1101/2023.06.07.544017|https://github.com/KociOrges/cviewer|
|SPIKEPIPE|accurate quantification of eukaryotic species occurrences and intraspecific abundance change using DNA barcodes or mitogenomes|10.1111/1755-0998.13057 |https://github.com/dougwyu/ArcDyn|
|CAMAMED|composition-aware mapping-based analysis of metagenomic data|10.1093/nargab/lqaa107|https://github.com/mhnb/camamed|
|Sunbeam|pipeline for analyzing metagenomic sequencing experiments|10.1186/s40168-019-0658-x|https://github.com/sunbeam-labs/sunbeam|
|ViWrap|identify, bin, classify, and predict viral–host relationships for viruses from metagenomes|10.1002/imt2.118|https://github.com/AnantharamanLab/ViWrap|
|BRAKER3|Fully Automated Genome Annotation Using RNA-Seq and Protein Evidence|10.1101/2023.06.10.544449|https://hub.docker.com/r/teambraker/braker3|
|pyGenomeViz|genome visualization python package for comparative genomics||https://pypi.org/project/pygenomeviz/|
|clinker|Gene cluster comparison figure generator|10.1093/bioinformatics/btab007|https://github.com/gamcil/clinker|
|BGCFlow|Systematic pangenome workflow for the analysis of biosynthetic gene clusters across large genomic datasets|10.1101/2023.06.14.545018|https://github.com/NBChub/bgcflow|
|mPTP|Species delimitation using the multi-rate Poisson Tree Processes|10.1093/bioinformatics/btx025|https://github.com/Pas-Kapli/mptp|
|ASAP|assemble species by automatic partitioning|10.1111/1755-0998.13281|https://bioinfo.mnhn.fr/abi/public/asap/|
|odp |Oxford Dot Plots or Ancient gene linkages (ALG)-genome comparisons|10.1038/s41586-023-05936-6|https://github.com/conchoecia/odp|
|GenAlEx |population genetic analysis|10.1093/bioinformatics/bts460|https://biology-assets.anu.edu.au/GenAlEx/Welcome.html|
|Bamboozle|identification and quantification of intraspecific barcodes|10.1101/2023.03.16.532925|https://github.com/topel-research-group/Bamboozle|
|MultiTax |Python package to obtain, parse and explore biological taxonomies||https://github.com/pirovc/multitax|
|euka|detection of eukaryotic taxa from modern and ancient environmental DNA using pangenomic reference graphs|10.1101/2023.04.04.535531|https://github.com/grenaud/vgan|
|Reconstructor|COBRApy-compatible tool that produces high-quality draft reconstructions|10.1093/bioinformatics/btad367|https://github.com/emmamglass/reconstructor|
|metaDMG| Fast and Accurate Ancient DNA Damage Toolkit for Metagenomic Data|10.1101/2022.12.06.519264|https://github.com/metaDMG-dev|
|Bakdrive|identifying a minimum set of bacterial species driving interactions across multiple microbial communities|10.1093/bioinformatics/btad236|https://gitlab.com/treangenlab/bakdrive|
|Hecatomb|'sacrifice' false-positive viral reads from metagenomes to enrich for true-positive viral reads|10.1101/2022.05.15.492003|https://github.com/shandley/hecatomb|
|Phynteny|Synteny-based annotation of bacteriophage genes||https://github.com/susiegriggo/Phynteny|
|CALOUR|exploratory and interactive microbiome analysis based on heatmaps|10.1128/msystems.00269-18|https://github.com/amnona/calour|
|pangene|construct a pangenome gene graph||https://github.com/lh3/pangene|
|Phyloformer|towards fast and accurate phylogeny estimation with self-attention networks|10.1101/2022.06.24.496975|https://github.com/lucanest/Phyloformer|
|AmpliDiff|An Optimized Amplicon Sequencing Approach to Estimating Lineage Abundances in Viral Metagenomes|10.1101/2023.07.22.550164|https://github.com/JaspervB-tud/AmpliDiff|
|Greengenes2|Greengenes2 unifies microbial data in a single reference tree|10.1038/s41587-023-01845-1|https://greengenes2.ucsd.edu/|
|uDANCE|Updating phylogenetic and phylogenomic trees using divide and conquer||https://github.com/balabanmetin/uDance|
|The LifeMonitor GitHub app|application of workflow sustainability best practices||https://crs4.github.io/life_monitor/lm_wft_best_practices_github_app|
|MetaCerberus|HMM-based implementation for robust functional annotation across the tree of life|10.1101/2023.08.10.552700|https://github.com/raw-lab/metacerberus|
|ggvolc| translates differential expression datasets and RNAseq data into informative volcano plots||https://github.com/loukesio/ggvolc|
|GEN-ERA toolbox|comparative genomics of bacteria and small eukaryotes|10.1093/gigascience/giad022|https://github.com/Lcornet/GENERA|
|MultiQC|Aggregate results from bioinformatics analyses across many samples into a single report|10.1093/bioinformatics/btw354|https://github.com/ewels/MultiQC|
|AMAW (Automated MAKER2 Annotation Wrapper)|automated gene annotation for non-model unicellular eukaryotic genomes|10.1101/2021.12.07.471566|https://bitbucket.org/phylogeno/amaw/src/master/|
|CRACOT|CRitical Assessment of genomic COntamination detection at several Taxonomic ranks|10.1101/2022.11.14.516442|https://github.com/Lcornet/GENERA/wiki/20.-CRACOT|
|ORPER (ORganism PlacER)|A Workflow for Constrained SSU rRNA Phylogenies|10.3390/genes12111741|https://github.com/Lcornet/ORPER|
|ToRQuEMaDA|tool for retrieving queried Eubacteria, metadata and dereplicating assemblies|10.7717/peerj.11348|https://bitbucket.org/phylogeno/tqmd/src/master/|
|BinaRena|interactive platform for human-guided exploration and binning of metagenomes|10.1186/s40168-023-01625-8|https://github.com/qiyunlab/binarena|
|DeGenPrime|robust primer design and optimization|10.1101/2023.08.11.553048|https://github.com/raw-lab/DeGenPrime|
|KOunt|a reproducible KEGG orthologue abundance workflow|10.1093/bioinformatics/btad483|https://github.com/WatsonLab/KOunt|
|iModulonDB|a knowledgebase of microbial transcriptional regulation derived from machine learning|10.1093/nar/gkaa810|https://imodulondb.org/|
|DataHarmonizer|data harmonization, validation, aggregation and analysis of pathogen genomics|10.1099/mgen.0.000908|https://github.com/cidgoh/DataHarmonizer|
|ggCaller|Accurate and fast graph-based pangenome annotation and clustering|10.1101/gr.277733.123|https://github.com/samhorsfield96/ggCaller|
|snpless-nf|time-course analysis with bacterial NGS whole-genome data||https://github.com/kullrich/snpless-nf|
|metagenome-based virus-to-microbe ratio (VMR) estimate (mVRM)|Metagenome-derived virus-microbe ratios|10.1038/s41396-023-01431-y|https://gitlab.com/anagtz/mvmrs|
|THAPBI PICT|a fast, cautious, and accurate metabarcoding analysis pipeline|10.7717/peerj.15648|https://github.com/peterjc/thapbi-pict/|
|SemiBin2|self-supervised contrastive learning leads to better MAGs|10.1093/bioinformatics/btad209|https://github.com/BigDataBiology/SemiBin/|
|MetaDecoder|clustering metagenomic contigs|10.1186/s40168-022-01237-8|https://github.com/liu-congcong/MetaDecoder/|
|MArVD2|a machine learning enhanced tool to discriminate between archaeal and bacterial viruses|10.1038/s43705-023-00295-9|https://bitbucket.org/MAVERICLab/marvd2/src/master/|
|‘SRS’ R Package|Normalization of Microbiome Data Using Scaling with Ranked Subsampling (SRS)|10.3390/app112311473|https://github.com/vitorheidrich/SRS_q2-srs_info|
|BugSigDB|a database for identifying unusual abundance patterns in human microbiome studies|10.1038/s41587-023-01872-y|https://github.com/waldronlab/BugSigDB|
|MVP|Multi-choice Viromics Pipeline|10.1128/msystems.00888-24|https://gitlab.com/ccoclet/mvp|
|Mycotools|An Automated and Scalable Platform for Comparative Genomics|10.1101/2023.09.08.556886|https://github.com/xonq/mycotools|
|Orchestrating Microbiome Analysis (OMA) with Bioconductor|||https://microbiome.github.io/OMA/index.html|
|Effective microbial richness (EMR)||10.1038/s43705-021-00033-z|https://zenodo.org/record/4837436|
|Strain Genome Explorer (StrainGE)|toolkit to track and characterize low-abundance strains in complex microbial communities|10.1186/s13059-022-02630-0|https://github.com/broadinstitute/strainge|
|cpnDB|chaperonin database||https://www.cpndb.ca/|
|PopPUNK|Fast and flexible bacterial genomic epidemiology |10.1101/gr.241455.118|https://github.com/bacpop/PopPUNK|
|euka|tetrapodic and arthropodic taxa detection from modern and ancient environmental DNA|10.1111/2041-210X.14214|https://github.com/grenaud/vgan|
|compleasm|faster and more accurate reimplementation of BUSCO|10.1093/bioinformatics/btad595|https://github.com/huangnengCSU/compleasm|
|ANCOM-BC|Analysis of Compositions of Microbiomes with Bias Correction|10.1038/s41467-020-17041-7|https://github.com/FrederickHuangLin/ANCOM-BC-Code-Archive|
|BLCA|Bayesian-based LCA taxonomic classification method|10.1186/s12859-017-1670-4|https://github.com/qunfengdong/BLCA|
|FAST (Feature-Assisted Segmenter/Tracker)|Tracking bacteria at high density|10.1371/journal.pcbi.1011524|https://mackdurham.group.shef.ac.uk/FAST_DokuWiki/dokuwiki/doku.php|
|AleRax|gene and species tree co-estimation and reconciliation|10.1101/2023.10.06.561091|https://github.com/BenoitMorel/AleRax|
|Qmatey|automated pipeline for fast exact matching-based alignment and strain-level taxonomic binning and profiling of metagenomes|10.1093/bib/bbad351|https://github.com/bodeolukolu/Qmatey|
|MetADAta MicrobiomE (MADAME)|user-friendly bioinformatic tool for data and metadata retrieval in microbiome research|10.1101/2023.10.14.562335|https://github.com/BiomeResearchTeam/MADAME|
|VIRUSBreakend|identifies viral DNA presence and genomic integration|10.1093/bioinformatics/btab343|https://github.com/PapenfussLab/VIRUSBreakend|
|Spae|Phage toolkit to detect phage candidates for phage therapy||https://github.com/linsalrob/spae|
|pimba|PIpeline for MetaBarcoding Analysis which allows the use of customized databases, as well as other reference databases|10.1007/978-3-030-91814-9_10|https://github.com/reinator/pimba|
|BIRDS|Biodiversity Information Review and Decision Support package for R||https://greenswayab.github.io/BIRDS/index.html|
|CodiMD|collaborate in real-time with markdown||https://github.com/hackmdio/codimd|
|Protist_barcode_MAG_correspondence| integrate information between metabarcoding and metagenomic datasets and explain how metabarcoding OTUs and MAGs are related with the underlying biological entities they are hypothesised to represent|10.1101/2023.10.09.561583|https://github.com/beaplab/Protist_barcode_MAG_correspondence|
|nf-core/taxprofiler|metagenomic taxonomic classification and profiling|10.1101/2023.10.20.563221|https://github.com/nf-core/taxprofiler|
|SOCfinder|genomic tool for identifying cooperative genes in bacteria|10.1099/mgen.0.001171|https://github.com/lauriebelch/SOCfinder|
|BarKeeper|web framework to assemble, analyze and manage DNA barcode data and metadata|10.1111/2041-210X.14047|https://github.com/DNA-BarKeeper/barkeeper , http://barkeeper.uni-muenster.de/|
|RasperGade16S|Prediction and correction of 16S rRNA Gene Copy Numners (GCN)|10.1038/s43705-023-00266-0|https://github.com/wu-lab-uva/RasperGade16S|
|RasperGade|Reconstructing Ancestral State under Pulsed Evolution in R by Gaussian Decomposition|10.1093/sysbio/syac016|https://github.com/wu-lab-uva/RasperGade|
|BacArena|Modeling Framework for Cellular Communities in their Environments|10.1371/journal.pcbi.1005544|https://bacarena.github.io/|
|SPIRE|Searchable, Planetary-scale mIcrobiome REsource|10.1093/nar/gkad943|https://spire.embl.de/|
|Geometric viability assay (GVA)|high-throughput and low-waste viability assay for microbes|10.1038/s41564-023-01513-9|https://www.colorado.edu/lab/chatterjeelab/geometric-viability-assay-gva|
|ggrepel|Avoid overlapping text labels in graphs||https://github.com/slowkow/ggrepel|
|Environmentally-Driven Edge Detection Program (EnDED)|Disentangling environmental effects in microbial association networks|10.1186/s40168-021-01141-7|https://github.com/InaMariaDeutschmann/EnDED|
|Extended local similarity analysis (eLSA) |network construction|10.1101/2021.07.13.452187|https://github.com/InaMariaDeutschmann/TemporalNetworkBBMO|
|Extended local similarity analysis (eLSA) |extends LSA to time series data with replicates|10.1186/1752-0509-5-S2-S15|http://meta.usc.edu/softs/lsa|
|Seasonal niche differentiation among closely related marine bacteria||10.1038/s41396-021-01053-2|https://github.com/adriaaulaICM/bbmo_niche_sea|
|benchdamic|benchmarking of differential abundance method for microbiome data|10.1093/bioinformatics/btac778|https://bioconductor.org/packages/release/bioc/html/benchdamic.html|
|LMAS (Last Metagenomic Assembler Standing)|evaluating metagenomic short de novo assembly methods through defined communities|10.1093/gigascience/giac122|https://github.com/B-UMMI/LMAS|
|ALDEx2 (ANOVA-Like Differential Expression tool)|Analysis Of Differential Abundance Taking Sample and Scale Variation Into Account|10.1186/2049-2618-2-15|https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html|
|zCompositions |Imputation of Zeros, Nondetects and Missing Data in Compositional Data Sets|10.1016/j.chemolab.2015.02.019|https://github.com/cran/zCompositions|
|robCompositions|Compositional Data Analysis|10.1002/9781119976462.ch25|https://cran.r-project.org/web/packages/robCompositions/index.html|
|WebVOWL|Web-based Visualization of Ontologies|10.3233/SW-150200|http://vowl.visualdataweb.org/webvowl.html|
|StrainPanDA|A strain analysis pipeline based on pangenome|10.1002/imt2.41|https://github.com/xbiome/StrainPanDA|
|SpaceMicrobe|detect microbial reads in 10X Visium Spatial Gene Expression data||https://github.com/bedapub/space-microbe|
|splicekit|splicing analysis from short-read RNA-seq|10.1101/2023.05.25.542256|https://github.com/bedapub/splicekit|
|Besca|single-cell transcriptomics analysis toolkit |10.1093/nargab/lqab102|https://bedapub.github.io/besca/|
|SCALA|multimodal analysis of single-cell Next Generation Sequencing data|10.1016/j.csbj.2023.10.032|https://scala.fleming.gr|
|AMDirT |AncientMetagenomeDir Toolkit|10.12688/f1000research.134798.1|https://github.com/SPAAM-community/AMDirT|
|AncientMetagenomeDir|community curated resource of lists of all published shotgun-sequenced ancient metagenome or microbial genome-level enriched samples and libraries||https://github.com/SPAAM-community/AncientMetagenomeDir|
|Provenance Replay|Facilitating bioinformatics reproducibility|10.1371/journal.pcbi.1011676|https://github.com/qiime2/provenance-lib|
|IAS Hotspot Model|running species distribution models for marine species||https://github.com/biomobst/IAS_hotspot_model|
|EMODnetWFS|Access EMODnet Web Feature Service data through R||https://github.com/EMODnet/EMODnetWFS|
|EMODnetWCS|interrogation of and access to EMODnet geographic raster data in R though the EMODnet Web Coverage Services||https://github.com/EMODnet/EMODnetWCS|
|Btrait|facilitates working with species density data, combined with species traits||https://github.com/EMODnet/Btrait|
|comspat|analyze within-community spatial organization using species combinations|10.1111/ecog.06216|https://github.com/jamestsakalos/comspat|
|MultiBarcodeTools|Easy selection of optimal primers for eDNA multi-metabarcoding|10.1002/edn3.499|https://multibarcode.k.u-tokyo.ac.jp/|
|MAGqual|standalone pipeline to assess the quality of metagenome-assembled genomes|10.1101/2023.12.13.571510|https://github.com/ac1513/MAGqual|
|Predicted Relative Metabolomic Turnover (PRMT)|determining metabolic turnover|10.1186/2042-5783-1-4||
|MMINP|A computational framework of microbemetabolite interactions-based metabolic profiles predictor based on the O2-PLS algorithm|10.1080/19490976.2023.2223349|https://github.com/YuLab-SMU/MMINP|
|METAGENassist |user-friendly, web-based analytical pipeline for comparative metagenomic studies|10.1093/nar/gks497|http://www.metagenassist.ca/METAGENassist/faces/Home.jsp|
|plotgardener|genomic data visualization package for R|10.1093/bioinformatics/btac057|https://github.com/PhanstielLab/plotgardener|
|g(r)osling|build interactive genomics visualizations with Gosling||https://github.com/gosling-lang/grosling|
|ggcoverage|visualize and annotate genome coverage for various NGS data|10.1186/s12859-023-05438-2|https://github.com/showteeth/ggcoverage|
|App-SpaM|Alignment-free phylogenetic placement algorithm based on Spaced-word Matches|10.1101/2020.10.19.344986|https://github.com/matthiasblanke/App-SpaM|
|MicroNiche|assessing microbial niche breadth and overlap from amplicon sequencing data|10.1093/femsec/fiaa131|https://cran.r-project.org/web/packages/MicroNiche/index.html|
|MIMt|A curated 16S rRNA reference database with less redundancy and higher accuracy at species-level identification|10.1101/2023.12.15.571851|https://mimt.bu.biopolis.pt/|
|metabaR|r package for the evaluation and improvement of DNA metabarcoding data quality|10.1111/2041-210X.13552|https://github.com/metabaRfactory/metabaR|
|argNorm|Antibiotic resistance gene (ARG) normalization by mapping to the antibiotic resistance ontology (ARO) by CARD|10.5204/rep.eprints.252448|https://github.com/BigDataBiology/argNorm|
|stratfreshDB |Comprehensive dataset of shotgun metagenomes from oxygen stratified freshwater lakes and ponds|10.1038/s41597-021-00910-1|https://figshare.scilifelab.se/articles/dataset/StratFreshDB_v1_0/13005311/2|
|STAMPS 2019 wiki|||https://github.com/mblstamps/stamps2019/wiki|
|POGENOM|determine allele frequencies in the metagenomes and calculate π and FST values|10.1038/s41396-021-00985-z|https://github.com/EnvGen/POGENOM|
|Population-structure|Contains scripts used in the study "Geographic population structure and distinct population dynamics of globally abundant freshwater bacteria"|10.1101/2023.07.13.548520|https://github.com/thr44pw00d/population-structure|
|Coracle|a machine learning framework to identify bacteria associated with continuous variables|10.1093/bioinformatics/btad749|http://www.micportal.org/coracle/index/|
|GSR-DB|a manually curated and optimized taxonomical database for 16S rRNA amplicon analysis|10.1128/msystems.00950-23|https://manichanh.vhir.org/gsrdb/|
|multiSMASH|A workflow and scripts for large-scale antiSMASH analyses|10.5281/zenodo.8276143|https://github.com/zreitz/multismash/|
|phytools 2.0|an updated R ecosystem for phylogenetic comparative methods|10.7717/peerj.16505|https://github.com/liamrevell/Revell.phytools-v2/|
|pangenome_rf|Pangenome analysis using random forests|10.1073/pnas.2304934120|https://github.com/alanbeavan/pangenome_rf|
|BactInt|domain driven transfer learning approach for extracting inter-bacterial associations from biomedical text|10.1016/j.compbiolchem.2023.108012||
|happi|a hierarchical approach to pangenomics inference|10.1186/s13059-023-03040-6|https://github.com/statdivlab/happi|
|VESPA (Vertebrate Eukaryotic endoSymbiont and Parasite Analysis)|primers and optimized protocol for eukaryotic endosymbiont metabarcoding|10.1038/s41467-023-44521-3||
|microTrait|Trait-Based Representation of Microbial Genomes|10.3389/fbinf.2022.918853|https://github.com/ukaraoz/microtrait|
|Metagenomic Thermometer|predicting environmental temperature based on metagenomic sequences|10.1093/dnares/dsad024|http://palaeo.nig.ac.jp/metathermo|
|Tome|prediction of optimal growth temperature of microorganisms and enzyme catalytic optima||https://github.com/EngqvistLab/Tome|
|CNN_tRNA_OGT|tRNA thermometer to estimate microbial adaptation to temperature|10.1093/nar/gkaa1030|https://bitbucket.org/bucklerlab/cnn_trna_ogt/src/master/|
|ASTRAL|genome-scale coalescent-based species tree estimation|10.1093/bioinformatics/btu462|https://github.com/smirarab/ASTRAL/|
|ASTRAL-III|polynomial time species tree reconstruction from partially resolved gene trees|10.1186/s12859-018-2129-y|https://github.com/smirarab/ASTRAL|
|AleRax|species & gene tree co-estimation and reconciliation |10.1101/2023.10.06.561091|https://github.com/BenoitMorel/AleRax|
|GeneRax|Species-Tree-Aware Maximum Likelihood-Based Gene  Family Tree Inference under Gene Duplication, Transfer, and Loss|10.1093/molbev/msaa141|https://github.com/BenoitMorel/GeneRax|
|SpeciesRax|Maximum Likelihood Species Tree Inference from Gene Family Trees under Duplication, Transfer, and Loss|10.1093/molbev/msab365|https://github.com/BenoitMorel/GeneRax|
|Asteroid|infer species trees from gene trees under high proportions of missing data|10.1093/bioinformatics/btac832|https://github.com/BenoitMorel/Asteroid|
|ALE|generic assembly likelihood evaluation framework for assessing the accuracy of genome and metagenome assemblies|10.1093/bioinformatics/bts723|http://www.alescore.org|
|AnnoView |large-scale analysis, comparison, and visualization of microbial gene neighborhoods|10.1101/2024.01.15.575735|http://annoview.uwaterloo.ca/annoview/|
|KAUST Metagenomic Analysis Platform (KMAP)|analyse (browse, query, compare) annotated metagenomic datasets|10.3389/fsci.2023.1038696|http://www.cbrc.kaust.edu.sa/kmap/index.php|
|phydms|phylogenetic analyses using deep mutational scanning data to inform the substitution models|10.7717/peerj.3657|https://jbloomlab.github.io/phydms/|
|Friends Don't Let Friends Make Bad Graphs|opinionated essay about good and bad practices in data visualization|10.5281/zenodo.7542491|https://github.com/cxli233/FriendsDontLetFriends|
|animalcules|R package for interactive microbiome analysis through either an interactive interface facilitated by R Shiny|10.1186/s40168-021-01013-0|https://github.com/wejlab/animalcules|
|eDITH|an R-package to spatially project eDNA-based biodiversity across river networks|10.1101/2024.01.16.575835|https://github.com/lucarraro/test_eDITH|
|edgeR 4.0|powerful differential analysis of sequencing data with expanded functionality and improved support for small counts and larger datasets|10.1101/2024.01.21.576131|https://bioinf.wehi.edu.au/edgeR/|
|NNforge|Predicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences|10.1101/2024.01.26.576649|https://gitlab.com/oveoyas/nnforge|
|DeepGOMeta|Predicting functions for microbes|10.1101/2024.01.28.577602|https://github.com/bio-ontology-research-group/deepgometa|
|SingleM|profiling shotgun metagenomes using conserved regions within universal marker genes|10.1101/2024.01.30.578060|https://wwood.github.io/singlem/|
|Sandpiper|interrogating public shotgun metagenome datasets, presenting the results generated by SingleM|10.1101/2024.01.30.578060|https://sandpiper.qut.edu.au|
|Smafa|A read aligner for aligned sequences||https://github.com/wwood/smafa|
|TEPI|Taxonomy-aware Embedding and Pseudo-Imaging for Scarcely-labeled Zero-shot Genome Classification|10.48550/arXiv.2401.13219||
|FEEDS (Food wastE biopEptiDe claSsifier)|biopeptide prediction and classification tool|10.1016/j.crbiot.2024.100186|https://github.com/vborincenturion/feeds|
|ClustVis|a web tool for visualizing clustering of multivariate data|10.1093/nar/gkv468|https://biit.cs.ut.ee/clustvis/|
|ReMIND|A constraint-based framework to reconstruct interaction networks in microbial communities|10.1101/2024.01.30.577913|https://github.com/EPFL-LCSB/remind|
|mkLTG|taxonomic assignment of metabarcoding sequences using variable identity thresholds|10.1007/s42977-024-00201-x|https://github.com/meglecz/mkLTG|
|COInr and mkCOInr|Building and customizing a nonredundant barcoding reference database from BOLD and NCBI|10.1111/1755-0998.13756|https://github.com/meglecz/mkCOInr|
|BASTA (BAsic Sequence Taxonomy Annotation)|Taxonomic classification of sequences and sequence bins using last common ancestor estimations|10.1111/2041-210X.13095|https://github.com/timkahlke/BASTA|
|GenomeFace|a deep learning-based metagenome binner|10.1101/2024.02.07.579326||
|microbeMASST|a taxonomically informed mass spectrometry search tool for microbial metabolomics data|10.1038/s41564-023-01575-9|https://github.com/robinschmid/microbe_masst|
|Dashing Growth Curves|web application for rapid and interactive analysis of microbial growth curves|10.1186/s12859-024-05692-y|https://dashing-growth-curves.ethz.ch/|
|mbctools|Metabarcoding and Cross-Platform Pipeline|10.1101/2024.02.08.579441|https://github.com/GuilhemSempere/mbctools|
|PIMENTA|PIpeline for MEtabarcoding through Nanopore Technology used for Authentication|10.1101/2024.02.14.580249|https://github.com/WFSRDataScience/PIMENTA|
|Mg-Traits pipeline|advancing functional trait-based approaches in metagenomics|10.3897/aca.4.e64908|https://github.com/pereiramemo/metagenomic_pipelines|
|Environmentally-Driven Edge Detection Program (EnDED)|Disentangling environmental effects in microbial association networks|10.1186/s40168-021-01141-7|https://github.com/InaMariaDeutschmann/EnDED|
|rshift|paleoecology and regime shift analysis||https://github.com/alexhroom/rshift|
|Placement Evaluation WOrkflows (PEWO)|a collection of workflows to benchmark phylogenetic placement|10.1093/bioinformatics/btaa657|https://github.com/phylo42/PEWO|
|EPIK|precise and scalable evolutionary placement with informative k-mers|10.1093/bioinformatics/btad692|https://github.com/phylo42/EPIK|
|PhylteR|Efficient Identification of Outlier Sequences in Phylogenomic Datasets|10.1093/molbev/msad234|https://github.com/damiendevienne/phylter|
|SpacerPlacer|reconstructing ancestral CRISPR spacer arrays along a given phylogenetic tree|10.1101/2024.02.20.581079|https://github.com/fbaumdicker/SpacerPlacer|
|PIGMENTUM|an easy pigment-based tool for monitoring phytoplankton community composition|10.3354/meps14518||
|dbBact | community knowledge base for information about mircobial ecology|10.1093/nar/gkad527|https://www.dbbact.org/main|
|ProkFunFind|detect genes of functional interest|10.1128/msystems.00036-24|https://github.com/nlm-irp-jianglab/ProkFunFind|
|phyloMDA|an R package for phylogeny-aware microbiome data analysis|10.1186/s12859-022-04744-5|https://github.com/liudoubletian/phyloMDA|
|sandbox.bio|Interactive bioinformatics tutorials||https://sandbox.bio/|
|awesome-r-dataviz|Curated resources for Data Visualization, Drawing & Publishing in R||https://krzjoa.github.io/awesome-r-dataviz|
|RAWGraphs|A free and open source tool for data visualization||https://www.rawgraphs.io/|
|TICI|a taxon-independent community index for eDNA-based ecological health assessment|10.7717/peerj.16963|https://zenodo.org/records/10396304|
|Microbiota data analysis|||https://scienceparkstudygroup.github.io/microbiome-lesson/|
|microViz | R package for analysis and visualization of microbiome sequencing data|10.21105/joss.03201|https://david-barnett.github.io/microViz/index.html|
|MicrobiomeStat|Supporting Longitudinal Microbiome Analysis in R||https://www.microbiomestat.wiki/|
|GToTree|a user-friendly workflow for phylogenomics|10.1093/bioinformatics/btz188|https://github.com/AstrobioMike/GToTree|
|Web of Microbes (WoM)| a curated microbial exometabolomics database for linking chemistry and microbes|10.1186/s12866-018-1256-y|https://metatlas.nersc.gov/wom/project-begin.view|
|GenomeSPOT|Predicting microbial growth conditions from amino acid composition|10.1101/2024.03.22.586313|https://github.com/cultivarium/GenomeSPOT|
|Kioloa PopGen|SNP-Based population genomics||https://green-striped-gecko.github.io/kioloa/|
|srahunter|A User-Friendly Tool for Efficient Retrieval and Management of SRA Data|10.1101/2024.03.19.585745|https://socket.dev/pypi/package/srahunter|
|Basestack |platform for rapid and real time analysis of Nanopore generated sequencing data||https://github.com/jhuapl-bio/Basestack|
|anpan|consolidate statistical methods for strain analysis||https://github.com/biobakery/anpan|
|MACARRoN|Metabolome Analysis and Combined Annotation Ranks to pRioritize Novel bioactives||https://huttenhower.sph.harvard.edu/macarron/|
|WAAFLE| identifying LGTs from community metagenomes||https://huttenhower.sph.harvard.edu/waafle|
|specificity|an R package for analysis of feature specificity|10.1186/s40793-022-00426-0|https://github.com/darcyj/specificity|
|FAIR training handbook|||https://elixir-europe-training.github.io/ELIXIR-TrP-FAIR-training-handbook/|
|EXPLANA|EXPLoratory ANAlysis and feature selection in cross-sectional and longitudinal microbiome studies|10.1101/2024.03.20.585968|https://github.com/JTFouquier/explana|
|ORCA|Predicting replication origins in circular prokaryotic chromosomes|10.1101/2024.03.28.587133| https://github.com/ZoyavanMeel/ORCA|
|hAMRonization|Enhancing antimicrobial resistance prediction using the PHA4GE AMR detection specification and tooling|10.1101/2024.03.07.583950|https://github.com/pha4ge/hAMRonization|
|QMD|quantify microbial absolute abundance differences between groups|10.1002/imt2.78|https://github.com/Xingyinliu-Lab/QMD/tree/master|
|PLASS|Protein-Level ASSembler|10.1038/s41592-019-0437-4|https://github.com/soedinglab/plass|
|PenguiN|Protein guided nucleotide assembler|10.1101/2024.03.29.587318|https://github.com/soedinglab/plass|
|LRBinner|Binning Error-Prone Long Reads Using Auto Encoders|10.1186/s13015-022-00221-z|https://github.com/anuradhawick/LRBinner|
|MGnify-LR WorkflowMGnify-Long Read Workflow||https://github.com/KateSakharova/mgnify-lr|
|R-package Tutorial|||https://github.com/loukesio/R-package-tutorial|
|BASALT|refines binning from metagenomic data and increases resolution of genome-resolved metagenomic analysis|10.1038/s41467-024-46539-7|https://github.com/EMBL-PKU/BASALT|
|CyanoCyc |cyanobacterial web portal|10.3389/fmicb.2024.1340413|https://cyanocyc.org/|
|Predictive Ecosystem Analyzer (PEcAn) |Develop and promote accessible tools for reproducible ecosystem modeling and forecasting||https://pecanproject.github.io/|
|slow5curl|Streamlining remote nanopore data access|10.1093/gigascience/giae016|https://github.com/BonsonW/slow5curl|
|Resistance Gene Association and Inference Network (ReGAIN)|Assessing Probabilistic Co-Occurrence Between Resistance Genes in Bacterial Pathogens|10.1101/2024.02.26.582197|https://github.com/ERBringHorvath/regain_cl|
|OlinkRPackage|versatile toolbox to enable easy and smooth handling of Olink NPX data to speed up your proteomic research||https://github.com/Olink-Proteomics/OlinkRPackage|
|radEmu|R package for estimating changes in the abundance of microbial taxa |10.48550/arXiv.2402.05231|https://github.com/statdivlab/radEmu|
|QuickVariants|Fast and Accurate Variant Identification Tool for Sequencing-Based Studies||https://github.com/caozhichongchong/QuickVariants|
|Functional resemblance|A new look at functional beta diversity|10.1101/2024.02.25.580632||
|The ASV registry|The ASV registry is part of the project GBOL III|10.3897/arphapreprints.e125475|https://asv.bolgermany.de/metabarcoding|
|GSMc|Global Soil Mycobiome consortium dataset|10.1007/s13225-021-00493-7|https://github.com/Mycology-Microbiology-Center/GSMc|
|happi| Hierarchical Approach to Pangenomics Inference|10.1186/s13059-023-03040-6|https://github.com/statdivlab/happi|
|breakaway|premier package for statistical analysis of microbial diversit||https://github.com/adw96/breakaway|
|Gene tRee Ordination Visualization for ExplorationS (groves)|R package for visualizing a set of gene-level phylogenies|10.1093/biostatistics/kxad025|https://github.com/statdivlab/groves|
|Count Regression for Correlated Observations with the Beta-binomial (corncob)|R package for modeling relative abundance and testing hypotheses about the effect of covariates on relative abundance|10.1214/19-AOAS1283|https://github.com/statdivlab/corncob|
|tinyvamp | package for estimation and removal of measurement error in high-throughput sequencing data|10.48550/arXiv.2204.12733|https://github.com/statdivlab/tinyvamp|
|Predicting Absolute and Relative Abundance by Modeling Efficiency to Derive Intervals and Concentrations (paramedic)|R package for estimating microbial concentration|10.1111/biom.13503|https://github.com/statdivlab/paramedic|
|Regression, Inference, and General Data Analysis Tools for R (rigr)|R package to streamline data analysis in R||https://github.com/statdivlab/rigr|
|global microbial smORF catalogue (GMSC) |integrated, consistently-processed, smORFs catalogue of the microbial world|10.1101/2023.12.27.573469|https://gmsc.big-data-biology.org/home|
|BIOlogical ocean data reforMATting Effort (BIO-MATE)|reformats openly sourced published datasets from oceanographic voyages|10.1038/s41597-024-03038-0|https://github.com/KimBaldry/BIOMATE-Rpackage|
|MicrobioSee|A Web-Based Visualization Toolkit for Multi-Omics of Microbiology|10.3389/fgene.2022.853612|https://microbiosee.gxu.edu.cn|
|Snekmer|scalable pipeline for protein sequence fingerprinting based on amino acid recoding|10.1093/bioadv/vbad005|https://github.com/PNNL-CompBio/Snekmer|
|DataHarmonizer|A standardized browser-based spreadsheet editor and validator that can be run offline and locally|10.1099/mgen.0.000908|https://github.com/cidgoh/DataHarmonizer|
|National Microbiome Data Collaborative (NMDC) Schema|FAIR microbiome data sharing network||https://github.com/microbiomedata/nmdc-schema|
|ZooTraits|R shiny app for exploring animal trait data for ecological and evolutionary research|10.1002/ece3.11334|https://github.com/thiago-goncalves-souza/zootraits|
|NORMAN ARB&ARG|NORMAN Antibiotic Resistant Bacteria and Resistance Genes Database|10.1016/j.watres.2024.121689|https://www.norman-network.com/nds/bacteria/|
|metaVaR|MetaVariant Species (MVS) for reference-free and metagenomic-based population genomic analysis|10.1371/journal.pone.0244637|https://github.com/madoui/metaVaR|
|Microbe Atlas Project|Explore the microbiome of the world||https://microbeatlas.org/|
|MAPseq|Highly efficient k-mer search with confidence estimates, for rRNA sequence analysis|10.1093/bioinformatics/btx517|https://github.com/jfmrod/mapseq|
|World Ocean Database (WOD)|||https://www.ncei.noaa.gov/products/world-ocean-database|
|Ocean Climate Laboratory Team (OCL) |||https://www.ncei.noaa.gov/products/ocean-climate-laboratory|
|Microbiome Analysis Pipeline (LCPM)| comprehensive pipeline for analyzing microbiome data from the LCMP cohort|10.1038/s41591-024-02963-2|https://github.com/raeslab/QMP-Microbiome-CRC-confounders/|
|Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT) | taxonomic classification of long DNA sequences and metagenome assembled genomes (MAGs / bins) |10.1186/s13059-019-1817-x|https://github.com/MGXlab/CAT_pack|
|Read Annotation Tool (RAT)| estimates the taxonomic composition of metagenomes using CAT and BAT output|10.1038/s41467-024-47155-1|https://github.com/MGXlab/CAT_pack|
|Viral Eukaryotic Bacterial Archaeal (VEBA)|modular bioinformatics suite for end-to-end genome-resolved prokaryotic, (micro)eukaryotic, and viral multi-omics|10.1101/2024.03.08.583560|https://github.com/jolespin/veba|
|microgeo|an R package rapidly displays the biogeographic traits of soil microbial communities on maps|10.1093/femsec/fiae087|https://github.com/ChaonanLi/microgeo|
|tidyomics |enhancing omic data analyses|10.1038/s41592-024-02299-2|https://github.com/tidyomics|
|Gut Microbiome Health Index (GMHI)|A predictive index for health status using species-level gut microbiome profiling|10.1038/s41467-020-18476-8|https://github.com/jaeyunsung/GMHI_2020|
|GMWI-webtool|a user-friendly browser application for assessing health through metagenomic gut microbiome profiling|10.1093/bioinformatics/btad061|https://gmwi-webtool.github.io/|
|microHI (microbiota health index) |health index of the environmental microbiota to predict the health status of aquatic wildlife|10.1016/j.ecolind.2023.110346||
|Distance to Reference Communities Index (DRCI)|tool to assess communities' restoration success|10.1111/rec.14194||
|Ocean Barcode Atlas|a web service to explore the biodiversity and biogeography of marine organisms|10.1111/1755-0998.13322|https://oba.mio.osupytheas.fr/ocean-atlas/|
|QuPath|Open Software for Bioimage Analysis|10.1038/s41598-017-17204-5|https://qupath.github.io|
|MANOCCA (Multivariate Analysis of Conditional CovAriance)|test for the effect of a predictor on the covariance matrix of a multivariate outcome|10.1101/2023.09.20.558234|https://gitlab.pasteur.fr/statistical-genetics/manocca|
|nf-core/nanoseq|Nanopore demultiplexing, QC and alignment pipeline|10.1038/s41587-020-0439-x|https://nf-co.re/nanoseq/3.1.0|
|ONTbarcoder 2.0|rapid species discovery and identification with real-time barcoding facilitated by Oxford Nanopore R10.4|10.1111/cla.12566|https://github.com/asrivathsan/ONTbarcoder|
|SeMPI|a genome-based secondary metabolite prediction and identification web server|10.1093/nar/gkx289|https://www.pharmaceutical-bioinformatics.de/SeMPI/|
|Thanos|An R Package for the Gene-Centric Analysis of Functional Potential in Metagenomic Samples|10.3390/microorganisms12071264|https://github.com/zhezhaozoe/thanos|
|Shiny app|Put your Shiny app on the web by using your own servers or Posit’s hosting service||https://shiny.posit.co/|
|gargammel|a sequence simulator for ancient DNA|10.1093/bioinformatics/btw670|https://grenaud.github.io/gargammel/|
|MALT|alignment of metagenomic reads against a database of reference sequences|10.1038/s41559-017-0446-6|https://software-ab.cs.uni-tuebingen.de/download/malt/welcome.html|
|SNPEvaluation|a tool allowing for evaluating and visualizing confidence in variant calling of low-coverage pathogen genomes|10.1073/pnas.1820447116|https://github.com/andreasKroepelin/SNP_Evaluation|
|MaltExtract-Interactive Plotting App (MEx-IPA) tool|interactive viewer of output from the HOPS metagenomic authentication pipeline|10.5281/zenodo.3380011|https://zenodo.org/records/4771942|
|EAGER |scalable and reproducible bioinformatics best-practise processing pipeline for genomic NGS sequencing data, with a focus on ancient DNA (aDNA) data|10.1186/s13059-016-0918-z|https://github.com/apeltzer/EAGER-GUI|
|nf-core/eager|Reproducible, portable, and efficient ancient genome reconstruction|10.7717/peerj.10947|https://github.com/nf-core/eager|
|cuperdec|R package to generate 'Cumulative Percent Decay' curves, with optional filtering functions, for microbial taxonomic profiles|10.1073/pnas.2021655118|https://github.com/jfy133/cuperdec|
|DamageProfiler|calculate damage profiles of mapped ancient DNA reads|10.1093/bioinformatics/btab190|https://github.com/Integrative-Transcriptomics/DamageProfiler|
|metaBIT|integrative and automated metagenomic pipeline for analysing microbial profiles from high-throughput sequencing shotgun data|10.1111/1755-0998.12546|https://bitbucket.org/Glouvel/metabit/src/master/|
|CoproID|predicts the source of coprolites and paleofeces using microbiome composition and host DNA content|10.7717/peerj.9001|https://github.com/nf-core/coproid|
|FEAST|fast expectation-maximization for microbial source tracking|10.1038/s41592-019-0431-x|https://github.com/cozygene/FEAST|
|HOPS|automated detection and authentication of pathogen DNA in archaeological remains|10.1186/s13059-019-1903-0|https://github.com/rhuebler/HOPS|
|Holi|pipeline for taxonomic profiling of ancient metagenomic reads based on competitive mapping to large databases|10.1038/nature19085|https://github.com/ancient-eDNA/Holi|
|BacDiveR |R package provides a programmatic interface to the Bacterial Diversity Metadatabase of the DSMZ ||https://github.com/TIBHannover/BacDiveR|
|Metapresence| accurate species detection in metagenomics based on the genome-wide distribution of mapping reads|10.1128/msystems.00213-24|https://github.com/davidesangui/metapresence|
|fqlink|retrieve ENA fastq links from NCBI accession codes||https://github.com/TTTPOB/fqlink|
|Global Change Analysis Model (GCAM)|integrated assessment tool for exploring consequences and responses to global change|10.5194/gmd-12-677-2019|https://github.com/JGCRI/gcam-core|
|Global Biosphere Management Model (GLOBIOM)|A global model to assess competition for land use between agriculture, bioenergy, and forestry||https://globiom.org/|
|DisBalance|a platform to automatically build balance-based disease prediction models and discover microbial biomarkers from microbiome data|10.1093/bib/bbab094|https://github.com/yangfenglong/DisBalance|
|Microbiome data science with R/Bioconductor|||https://microbiome.github.io/course_2022_radboud/|
|gcplyr| import, wrangle, and do model-free analyses of microbial growth curve data|10.1186/s12859-024-05817-3|https://mikeblazanin.github.io/gcplyr/|
|Ocean Monitoring Indicators (OMI)|free downloadable trends and datasets covering the past quarter of a century||https://marine.copernicus.eu/access-data/ocean-monitoring-indicators|
|GLMMs for the Microbiome|Applying generalized linear mixed models (GLMMs) to wild metabarcoding data|10.1128/msystems.00040-23|https://arsweeny.github.io/microbiome-glmm/|
|Island biogeography theory and the gut microbiome||10.1093/ismejo/wrae114|https://github.com/Gibbons-Lab/IBT-and-the-Gut-Microbiome|
|Shiny-phyloseq| interactive web application that provides a graphical user interface to the phyloseq|10.1093/bioinformatics/btu616|https://joey711.github.io/shiny-phyloseq/|
|phold|Phage Annotation using Protein Structures||https://github.com/gbouras13/phold|
|Prophage-DB| a database of prophages, their proteins, and associated metadata |10.1101/2024.07.11.603044|https://github.com/AnantharamanLab/Prophage-DB|
|mettannotator|a comprehensive and scalable Nextflow annotation pipeline for prokaryotic assemblies|10.1101/2024.07.11.603040|https://github.com/EBI-Metagenomics/mettannotator|
|SRA-Explorer|Mini web application to explore the NCBI Sequence Read Archive and easily access downloads for data||https://github.com/ewels/sra-explorer|
|Biodiversity of Marine Planktonic Copepods |Thesaurus (from the cited literature from 1883 to 2019)||https://copepodes.obs-banyuls.fr/en/|
|Phylo-tree_eDNA|create phylogenetic trees based on taxonomic assignments of Environmental DNA (eDNA) data||https://github.com/Enorya/Phylo-tree_eDNA|
|RiboGrove tools|a database of full-length prokaryotic 16S rRNA genes derived from completely assembled genomes|10.1016/j.resmic.2022.103936|https://github.com/masikol/ribogrove-tools|
|HiOrCo|Computing Higher-Order Co-occurrence patterns in microbial samples||https://github.com/cdanielmachado/HiOrCo|
|Dbfetch|Dbfetch provides an easy way to retrieve entries from various databases at the EMBL-EBI in a consistent manneρ|10.1093/nar/gkac240|https://www.ebi.ac.uk/Tools/dbfetch/|
|SNIPe|Selecting Novel Informative Primer-sets for e-DNA|10.1002/edn3.590|https://snipe.dlougheed.com|
|pipecraft|a Graphical User Interface (GUI) software that implements various popular tools for metabarcoding data analyses||https://github.com/pipecraft2/pipecraft|
|SURPI (Sequence-Based Ultra-Rapid Pathogen Identification)|pathogen identification from complex metagenomic next-generation sequencing (NGS) data|10.1101/gr.171934.113|https://github.com/chiulab/surpi/tree/master|
|CLARK|Fast, accurate and versatile sequence classification system|10.1186/s12864-015-1419-2|http://clark.cs.ucr.edu/|
|GOTTCHA (Genomic Origin Through Taxonomic CHAllenge)|Accurate read-based metagenome characterization using a hierarchical suite of unique signatures|10.1093/nar/gkv180|https://lanl-bioinformatics.github.io/GOTTCHA/|
|NanoGalaxy|Nanopore long-read sequencing data analysis in Galaxy|10.1093/gigascience/giaa105|https://nanopore.usegalaxy.eu|
|ASaiM|a Galaxy-based framework to analyze microbiota data|10.1093/gigascience/giy057|https://github.com/ASaiM/framework|
|MetaPro|a scalable and reproducible data processing and analysis pipeline for metatranscriptomic investigation of microbial communities|10.1186/s40168-023-01562-6|https://github.com/ParkinsonLab/MetaPro|
|NanoForms|an integrated server for processing, analysis and assembly of raw sequencing data of microbial genomes|10.7717/peerj.13056|https://nanoforms.tech/home/|
|MicrobiomePhylo|Metabarcoding Data Downstream Analysis |10.1101/2024.04.08.588598|https://microbiomephylo.com|
|lsaBGC-Pan| mine the pan-BGC-ome of a microbial taxon for biosynthetic golden nuggets|10.1099/mgen.0.000988|https://github.com/Kalan-Lab/lsaBGC-Pan|
|MICROPHERRET|MICRObial PHEnotypic tRait ClassifieR using Machine lEarning Techniques|10.1186/s40793-024-00600-6|https://github.com/BizzoTL/MICROPHERRET/|
|PopPUNK (POPulation Partitioning Using Nucleotide Kmers)|Fast and flexible bacterial genomic epidemiology|10.1101/gr.241455.118|https://github.com/bacpop/PopPUNK|
|PopPIPE (Population analysis PIPEline) |Downstream analysis of PopPUNK results. Produces subclusters and visualisations of all strains||https://github.com/bacpop/PopPIPE|
|ggCaller| Accurate and fast graph-based pangenome annotation and clustering|10.1101/2023.01.24.524926|https://github.com/bacpop/ggCaller|
|BacQuerya|search engine for genomic metadata for bacterial pathogens||https://github.com/bacpop/BacQuerya|
|mandrake|Fast visualisation of the population structure of pathogens using Stochastic Cluster Embedding|10.1098/rstb.2021.0237|https://github.com/bacpop/mandrake|
|unitig-caller|Determines presence/absence of sequence elements in bacterial sequence data|10.1101/695338|https://github.com/bacpop/unitig-caller|
|pyseer|a comprehensive tool for microbial pangenome-wide association studies|10.1093/bioinformatics/bty539|https://github.com/mgalardini/pyseer|
|CELEBRIMBOR (Core ELEment Bias Removal In Metagenome Binned ORthologs)|automatically generate pangenomes from metagenome assembled genomes|10.1093/bioinformatics/btae542|https://github.com/bacpop/CELEBRIMBOR|
|panaroo|A Bacterial Pangenome Analysis Pipeline||https://github.com/gtonkinhill/panaroo|
|Intermediate Research Programming Course|||https://iprogramming.bacpop.org/|
|GlobDB genomes database|a dereplicated set of species representative microbial genomes||https://globdb.org/|
|SMETANA (species metabolic interaction analysis)|describe the potential for cross-feeding interactions between community members|10.1073/pnas.1421834112|https://github.com/cdanielmachado/smetana|
|VITCOMIC2|visualization tool for the phylogenetic composition of microbial communities based on 16S rRNA gene amplicons and metagenomic shotgun sequencing|10.1186/s12918-018-0545-2|http://vitcomic.org/|
|The Microbe Directory (TMD)|database of microbial features and ecological characteristics||https://themicrobedirectory.shinyapps.io/website/|
|LEA (Latent Environment Allocation of microbial community data)|evaluate typicality and heterogeneity of microbial communities|10.1371/journal.pcbi.1006143|http://leamicrobe.jp/|
|metaGEM|reconstruction of genome scale metabolic models directly from metagenomes|10.1093/nar/gkab815|https://github.com/franciscozorrilla/metaGEM|
|NEMETEX (NEtwork for METabolic EXchanges)|graphical visualizations of metabolite exchanges within complex communities|10.1101/2022.10.19.492777|https://github.com/palakela/NEMETEX|
|coidb|A small python package for downloading and processing metabarcoding references||https://github.com/biodiversitydata-se/coidb|
|ARETE (Antimicrobial Resistance: Emergence, Transmission, and Ecology)|pipeline for profiling the genomic repertoire and evolutionary dynamics of microorganisms with a particular focus on pathogens|10.1099/mgen.0.000880|https://github.com/beiko-lab/arete|
|PHORCE (Phage-Host Observation for Rate estimation from Collapse Events)|phage amplification rates and phage-antibiotic interactions|10.1101/2024.06.07.597930||
|MicroEcoTools|Theoretical Microbial Ecology Computational Tools|10.1101/2024.08.19.608598|https://www.github.com/Soheil-A-Neshat/MicroEcoTools|
|Chronospaces|statistical exploration of divergence times|10.1111/2041-210X.14404|https://github.com/mongiardino/chronospace|
|FASTGA|Fast Genome Aligner||https://github.com/thegenemyers/FASTGA|
|binchicken|targeted recovery of low abundance metagenome assembled genomes through intelligent coassembly||https://github.com/AroneyS/binchicken|
|SEPP (SATé-Enabled Phylogenetic Placement)|phylogenetic placement of short reads into reference alignments and trees||https://github.com/smirarab/sepp/tree/master|
|TIPP (Taxonomic Identification and Phylogenetic Profiling)|a modification of SEPP for classifying query sequences (i.e. reads) using phylogenetic placement|10.1093/bioinformatics/btu721|https://github.com/TeraTrees/TIPP/|
|plateplanner|Design Beautiful Platemaps||https://plateplanner.nitro.bio/|
|miaverse|R/Bioconductor framework for microbiome data science||https://microbiome.github.io/|
|metabaRpipe|an R package to process raw metabarcoding data into a phyloseq object using a single command||https://github.com/fconstancias/metabaRpipe/|
|Microbial Abundances Retrieved from Sequencing data—automated NCBI Taxonomy (MARS)|processing of metagenomic datasets, taxonomic name resolution, and mapping to resources|10.1093/bioadv/vbae068|https://github.com/ThieleLab/mars-pipeline|
|GlobTherm| global database on thermal tolerances for aquatic and terrestrial organisms|10.1038/sdata.2018.22|https://datadryad.org/stash/dataset/doi:10.5061/dryad.1cv08|
|BOLD v4|A Centralized Bioinformatics Platform for DNA-Based Biodiversity Data|10.1007/978-1-0716-3581-0_26|https://v4.boldsystems.org|
| DENSE (DE Novo emerged gene SEarch)|detection of taxonomically restricted genes (TRGs) through phylostratigraphy, and filtering of TRGs for de novo emerged genes via genome comparisons and synteny search|10.1093/gbe/evae159|https://github.com/i2bc/dense|
|Big Fantastic Virus Database (BFVD) |repository of predicted viral protein structures|10.1101/2024.09.08.611582| https://bfvd.steineggerlab.workers.dev|
|MagicLamp|toolkit for annotation of genomic data using discreet and curated HMM sets||https://github.com/Arkadiy-Garber/MagicLamp|
|SprayNPray|Rapid and simple taxonomic profiling of genome and metagenome contigs|10.1186/s12864-022-08382-2|https://github.com/Arkadiy-Garber/SprayNPray|
|tombRaider |improved species and haplotype recovery from metabarcoding data through artefact and pseudogene exclusion|10.1101/2024.08.23.609468|https://github.com/gjeunen/tombRaider_R|
|SVbyEye|visual tool to characterize structural variation among whole genome assemblies|10.1101/2024.09.11.612418|https://github.com/daewoooo/SVbyEye|
|Fungtion|A Server for Predicting and Visualizing Fungal Effector Proteins|10.1016/j.jmb.2024.168613|https://step3.erc.monash.edu/Fungtion/|
|GDPF| distribution of prokaryotic protein families across the global biosphere|10.1093/nar/gkad869|http://bioinfo.qd.sdu.edu.cn/GDPF/|
|NAMERS|a purpose-built reference DNA sequence database to support applied eDNA metabarcoding|10.1101/2023.10.06.561210|https://namers.ca/|
|rix|Reproducible Data Science Environments with 'Nix'|	10.32614/CRAN.package.rix|https://github.com/ropensci/rix|
|SIAMCAT|pipeline for Statistical Inference of Associations between Microbial Communities And host phenoTypes|10.1186/s13059-021-02306-1|https://siamcat.embl.de/|
|MrIML 2.0|Multivariate (multi-response) interpretable machine learning|10.22541/au.172676147.77148600/v1|https://nfj1380.github.io/mrIML/index.html |
|WoRMS-functional-groups|access and format functional group data for marine species||https://github.com/tomjwebb/WoRMS-functional-groups|
|OceanOmics-amplicon-nf|Pipeline to analyse amplicon data from raw reads to ecologically informative phyloseq objects||https://github.com/MinderooFoundation/OceanOmics-amplicon-nf|
|CODARFE (COmpositional Data Anlises with Recursive Feature Elimination)| sparse compositional microbiome-predictors selection and prediction of continuous environmental factors|10.1101/2024.07.18.604052|https://github.com/alerpaschoal/CODARFE|
|Domestic Feral Cat Metagenomics|Functional insights into the effect of feralisation on the gut microbiota of cats worldwide|10.1101/2024.09.04.611329|https://alberdilab.github.io/domestic_feral_cat_metagenomics/|
|EasyMicrobiomeR |The best practice for microbiome analysis using R|10.1093/procel/pwad024|https://github.com/taowenmicro/EasyMicrobiomeR|
|GECCO|Biosynthetic Gene Cluster prediction with Conditional Random Fields|10.1101/2021.05.03.442509|https://gecco.embl.de/|
|A-microbiologist-wants-to-compute|||https://github.com/SvetlanaUP/A-microbiologist-wants-to-compute|
|||||
|||||
|||||
|||||
|||||

Tools image: <a href="http://www.freepik.com">Designed by vectorpouch / Freepik</a>


---